
<!DOCTYPE html PUBLIC
  "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" lang="en">

    
    
    
    
    



<!-- Mirrored from icbp.nci.nih.gov/icbp-centers/grants/ by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 29 Sep 2016 19:35:43 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />


    <link rel="stylesheet" type="text/css" href="includes/interactive_map.css" />
    <script src="../../../ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
    <script type="text/javascript" src="includes/interactive_map.js"></script>

    
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />

    
        <base  /><!--[if lt IE 7]></base><![endif]-->
    

    
        
  
    <link rel="stylesheet" type="text/css" href="../../portal_css/ICBP%20Theme/base-cachekey5221.css" />
        <!--[if lt IE 8]>    
    
    <link rel="stylesheet" type="text/css" media="screen" href="http://icbp.nci.nih.gov/portal_css/ICBP%20Theme/IEFixes-cachekey6323.css" />
        <![endif]-->
    
    <style type="text/css" media="screen">@import url(../../portal_css/ICBP%20Theme/resourceplone.formwidget.autocompletejquery.autocomplete-cachekey0423.css);</style>
    <link rel="stylesheet" type="text/css" media="screen" href="../../portal_css/ICBP%20Theme/resourceplone.app.dexterity.overlays-cachekey7584.css" />
    <style type="text/css" media="screen">@import url(../../portal_css/ICBP%20Theme/resourcetheme.icbp.stylesheetsmain-cachekey7250.css);</style>
    <link rel="stylesheet" type="text/css" media="all" href="../../portal_css/ICBP%20Theme/ploneCustom-cachekey8312.css" />

  
    <link rel="stylesheet" data-rel="kinetic-stylesheet" type="text/kss" href="../../portal_kss/ICBP%20Theme/at-cachekey5785.css" />
  
    <script type="text/javascript" src="../../portal_javascripts/ICBP%20Theme/jquery-cachekey2624.js"></script>
    <script type="text/javascript" src="../../portal_javascripts/ICBP%20Theme/jquery-integration-cachekey3788.js"></script>
       <!--[if lt IE 8]>
     
    <script type="text/javascript" src="http://icbp.nci.nih.gov/portal_javascripts/ICBP%20Theme/iefixes-cachekey1480.js"></script>
       <![endif]-->
     
    <script type="text/javascript" src="../../portal_javascripts/ICBP%20Theme/resourceplone.formwidget.autocompletejquery.autocomplete.min-cachekey0100.js"></script>





<link rel="kss-base-url" href="index.html" />
<title>ICBP Cooperative U01 Grants &mdash; Site</title>
        

    <link rel="shortcut icon" type="image/x-icon" href="../../favicon.ico" />
    <link rel="apple-touch-icon" href="../../touch_icon.png" />



    <link rel="home" href="../../index.html" title="Front page" />

    <link rel="contents" href="../../sitemap.html" title="Site Map" />






    <link rel="search" href="../../search_form.html" title="Search this site" />



        
        
        
        
        

        <meta name="viewport" content="width=device-width; initial-scale=0.6666; maximum-scale=1.0; minimum-scale=0.6666" />
        <meta name="generator" content="Plone - http://plone.org" />
    
</head>

<body class="template-document_view portaltype-document site-icbp section-icbp-centers icons-on" dir="ltr">

<div id="visual-portal-wrapper">

<div id="logos">
   <div id="left"><img src="../../nci.jpg" alt="" title="" height="43" width="286" /></div>
   <div id="right"><img src="../../nih.jpg" alt="" title="" height="43" width="322" /></div>
</div>
 
        <div id="portal-top" class="row">
                <div class="cell width-full position-0">
                    <div id="portal-header">
    <p class="hiddenStructure">
  <a accesskey="2" href="index.html#content">Skip to content.</a> |

  <a accesskey="6" href="index.html#portal-globalnav">Skip to navigation</a>
</p>

<div id="portal-personaltools-wrapper">

<h5 class="hiddenStructure">Personal tools</h5>





</div>



<div id="portal-searchbox">
    <form name="searchform" id="searchGadget_form" action="http://icbp.nci.nih.gov/search">

        <div class="LSBox">
        <label class="hiddenStructure" for="searchGadget">Search Site</label>

        <input name="SearchableText" type="text" size="18" title="Search Site" accesskey="4" class="searchField inputLabel" id="searchGadget" />

        <input class="searchButton" type="submit" value="Search" />

        <div class="searchSection">
            <input id="searchbox_currentfolder_only" class="noborder" type="checkbox" name="path" value="/icbp/icbp-centers" />
            <label for="searchbox_currentfolder_only" style="cursor: pointer">
                only in current section
            </label>
        </div>

        <div class="LSResult" id="LSResult" style=""><div class="LSShadow" id="LSShadow"></div></div>
        </div>
    </form>

    <div id="portal-advanced-search" class="hiddenStructure">
        <a href="../../search_form.html" accesskey="5">
            Advanced Search&hellip;
        </a>
    </div>

</div>
<a id="portal-logo" title="Site" accesskey="1" href="../../index.html">
    <img src="../../logo.png" alt="Site" title="Site" height="100" width="500" /></a>

    <h5 class="hiddenStructure">Sections</h5>

    <ul id="portal-globalnav">
<li id="portaltab-index_html" class="plain"><div class="left"></div><a href="../../index.html" title="">Home</a><div class="right" /></li>
<li id="portaltab-about" class="plain"><div class="left"></div><a href="../../about.html" title="">About ICBP</a><div class="right" /></li>
<li id="portaltab-icbp-centers" class="selected"><div class="left"></div><a href="../../icbp-centers.html" title="">ICBP Centers</a><div class="right" /></li>
<li id="portaltab-resources" class="plain"><div class="left"></div><a href="../../resources.html" title="">Resources</a><div class="right" /></li>
<li id="portaltab-icbp-news" class="plain"><div class="left"></div><a href="../../icbp-news.html" title="">ICBP News</a><div class="right" /></li>
<li id="portaltab-education-and-outreach" class="plain"><div class="left"></div><a href="../../education-and-outreach.html" title="">Education and Outreach</a><div class="right" /></li>

</ul>

</div>

                </div>
        </div>

        
        


        
    <div id="portal-columns" class="row">

        <div id="portal-column-content" class="cell width-full position-0">

            <div id="viewlet-above-content"><div id="portal-breadcrumbs">

    <span id="breadcrumbs-you-are-here">You
are here:</span>
    <span id="breadcrumbs-home">
        <a href="../../index.html">Home</a>
        <span class="breadcrumbSeparator">
            &rsaquo;
            
        </span>
    </span>
    <span id="breadcrumbs-1" dir="ltr">
        
            <a href="../../icbp-centers.html">ICBP Centers</a>
            <span class="breadcrumbSeparator">
                &rsaquo;
                
            </span>
            
         
    </span>
    <span id="breadcrumbs-2" dir="ltr">
        
            
            
            <span id="breadcrumbs-current">ICBP Cooperative U01 Grants</span>
         
    </span>

</div>
</div>

            
                <div class="">

                    

                    

    <dl class="portalMessage info" id="kssPortalMessage" style="display:none">
        <dt>Info</dt>
        <dd></dd>
    </dl>



                    
                    <table cellpadding="0" cellspacing="0" border="0" id="contentTable">
                    <tr>
                        <td valign="top">
                            <div id="content" style="">
                                
    
                                
    
                                 <div id="viewlet-above-content-title"></div>
                                 
                                     
                                     
            
                <h1 id="parent-fieldname-title" class="documentFirstHeading">
                    ICBP Cooperative U01 Grants
                </h1>
            
            
        
                                 
                                 <div id="viewlet-below-content-title">
</div>
    
                                 
                                     
                                     
            
                
            
            
        
                                 
    
                                 <div id="viewlet-above-content-body">

</div>
                                 <div id="content-core">
                                     
    
        
        <div class="" id="parent-fieldname-text-1103c264-6645-4417-b810-ad877acf6047">
            <p><strong>Johns Hopkins University<br />
Analysis of the Signaling and Mechanical Cues Promoting Invasion in Melanoma<br />
PIs:  Andre Levchenko, Douglas Lauffenburger<br />
Project Abstract:<br /></strong>
Metastatic melanoma is the deadliest form of skin cancer and one of the most aggressive cancer types, capable of rapid spreading out of the primary site. This proposal is aimed at analyzing the combined effect of changes in the genetic and signaling states of melanoma cells and the cell micro-environment on the degree of cell aggressiveness and putative metastatic potential. The main hypothesis underlying our analysis is that aggressiveness of melanoma cells is determined by a combination of different factors, with a major contribution from the state of cell microenvironment, including the structure of the ECM and the presence of soluble chemical factors. Alteration in the cell micro-environment, such as alignment of ECM fibers, can permit an otherwise non-metastatic cell to become more aggressive, whereas the aggressiveness of ostensibly metastatic cell can be moderated by environmental changes and targeted intervention in regulation of signaling cascades. We also hypothesize that cells can actively interact with the micro-environment not only by secreting or degrading ECM components, but by actively exercising forces on and thus aligning ECM fibers in the direction of a nearest anchor point, including the vasculature. This can in turn generate more anisotropic forces allowing cells to deform ECM even more and ultimately take advantage of the re-aligned ECM structure to migrate towards blood vessels. We will test these hypotheses in close collaboration with ICBP@MIT, first taking advantage of the new methodologies using shRNA libraries to perturb genetic targets (Aim 1) along with a broad range of environmental perturbations and then use the state of the art statistical analysis tools available at ICBP(gMIT to reduce complexity of resulting datasets, linking multiple environmental and genetic perturbation to aggressive cell behavior (Aim 2), We will use several novel assays in the process, increasing manageability of data acquisition and processing, as well as enhancing the biomimetic nature of experimentation. Finally, we will test the predictions arising from the models developed in Aim 2 through a series of tests, ranging from cell culture to tissue construct to model tissue levels (Aim 3). We anticipate that the proposed research will reveal new, previously unanticipated interplay between environmental and genetic factors in control of metastatic events in melanoma and potentially suggest new ways of battling this deadly disease. The results might also have relevance to other metastatic cancers.</p>

<p><strong>Memorial Sloan-Kettering Cancer Center<br />
Dissecting the Role of Alternative 3'UTRs in Cancer with 3'-seq<br />
PIs:  Christine Mayr, Christina S. Leslie<br />
Project Abstract: <br /></strong>
Recently, we discovered a new mechanism for cancer pathogenesis that does not require mutations in the DNA sequence. We found widespread expression of mRNAs with shorter 3' untranslated regions (3'UTRs) in cancer cells which was due to recognition of alternative cleavage and polyadenylation signals (ApA). The shortening of 3'UTRs of oncogenes led to increased protein levels which was sufficient for oncogenic transformation. However, shortening of 3'UTRs was also seen during physiological processes such as activation of T cells. In the long term, we want to find the mechanism for the generation of mRNAs with different 3'UTRs and determine the overall functional consequences of differential expression of 3'UTR isoforms for cancer. We will start by defining all mRNA 3' ends of the transcriptome in isogenic pairs of normal and transformed cells as well as in early and late stages of tumor development to identify the cancer-specific changes in 3'UTR expression. To do so, we will apply next-generation sequencing technology using our own protocol (3'-seq) to map mRNA 3' ends quantitatively. To start to address the functional consequences of mRNAs with shorter and longer 3'UTRs, we will use 3'-seq to determine the mRNA stability rates for all alternative mRNA isoforms to discover new (in)stability elements in 3'UTRs. To begin to identify causal factors for the shift in ApA signal choice, we will examine chromatin architecture at the 3'ends of transcription units since RNA processing occurs co-transcriptionally and in close proximity to RNA polymerase II and chromatin. We have preliminary data that histone variants and chromatin-associated factors influence ApA signal choice. Chromatin immunoprecipitation data and 3'-seq data will be correlated to describe the marks that are associated with the canonical 3' end of a gene and whose presence is altered upon expression of alternative 3' ends of mRNAs in activated and cancer states. This study will elucidate an important new mechanism of cancer pathogenesis. Better understanding of this mechanism may lay the groundwork for reversing this cancer-related pathway by therapeutic intervention. </p>

<p><strong>Moffitt Cancer Center<br />
Title:  Predicting Prostate Cancer Aggressiveness<br />
PIs:  Alexander R.A. Anderson, Gustavo Ayala, Simon W. Hayward<br />
Project Abstract:<br /></strong>
Cancer is a complex disease that is driven by interactions between tumor cells but also stromal cells and the microenvironment. We hypothesize that the interaction between the different cellular components of the tumor and the molecular signaling networks within each cell can delineate aggressive prostate cancer. We have selected intracellular and extracellular pathways and cell types that are representative of fundamental processes in human prostate cancer. We will use data from a large cohort of prostate cancer patients. These inputs, derived using state of the art methodology, and provided on a cell-per-cell basis will be used to derive a multi-scale mathematical model. This wealth of human data has not previously been achievable and will serve to both parameterize (on multiple scales) and validate the model. Mathematical modeling will generate a library of network triplets (i.e. intracellular signaling networks for tumor epithelium, normal stroma and reactive stroma) whose interactions can describe patient outcome. Networks will be selected using a genetic algorithm to identify and fix the fittest triplets. Triplets will then be selected based upon their ability to reflect invasive or non-invasive disease over a biologically relevant time period and subject to triage based upon their representation of histochemical characteristics. The most representative triplets will be validated against biological endpoints using in vivo experimentation. The validation phase will recapitulate key elements of the mathematical model in vivo to identify those models most functionally-relevant to human disease. These will be tested in vivo to make predictions that confirm or refute these results in silico. The most robust models (those that pass the testing and validation phases) will be compared to a test cohort of human clinical samples to correlate their characteristics against survival endpoints. Our unique combination of resources and team expertise represents an unparalleled environment providing a synergistic approach to understand prostate cancer beyond the limitations of currently applied scientific methodology. Our models begin and end with human data, assuring that the final products will provide new understanding of human prostate cancer.</p>

<p><strong>Stanford University<br />
Integrated Genomic Discovery and Functional Validation of Colorectal Cancer Loci<br />
PIs:  Calvin Kuo, Ji Hanlee<br />
Project Abstract:<br /></strong>
Colorectal carcinoma (CRC) arises from multiple mutations and genomic aberrations in distinct driver cancer genes that that in concert to spur neoplastic development and phenotype. This inherent genetic complexity greatly complicates both personalized diagnosis and treatment. Previously published studies have confined that large numbers of genes will be mutated or subject to genomic aberrations in CRC. A significant and emerging challenge for the post-genomic era is to identify which of these mutated genes are "driver" loci that functionally drive colon cancer development, versus "passenger" loci without functional relevance. Finally, it is of the highest priority that one forges these genetic observations with correlations of prognosis and clinical outcome. This can only be done if we better understand the unified biological ramifications of the combined and diverse multigenic driver background which act synergistically to promote CRC tumorigenesis. This proposal details an integrated analysis that will rely on the CRC genomic data generated by the Cancer Genome Atlas Project (TCGA) to discover novel candidate CRC genes and study multigenic CRC driver gene co-mutated / dysregulated modules within the genetic context of other drivers and provide biological validation in a powerful in vitro primary culture CRC model which can be engineered for multiple genetic events. To accomplish these goals, we will develop and implement novel statistical methodologies for the integrative analysis of multiple TCGA genomic and clinical data sets. The goal is to identify and prioritize novel CRC genes either singly or as co-mutated modules in combination with other known "driver" CRC genes. We will use the rich TCGA data set to conduct an integrated CRC genomic analysis of point mutations, gene expression, copy number aberrations and methylation data. We will prioritize the discovery of mutations and other genomic aberrations of these novel CRC genes that are associated with specific clinical stages of disease and other clinical parameters. These statistical and computational studies will then be directly coupled to rapid and robust functional target validation of candidate loci using our rigorously characterized in vitro primary intestinal culture methodology (Gotani et al, Nature Medicine, 2009), in which we have recently established the transforming activity of established CRC loci such as APC, KRAS and TP53. Genetic deletion and retroviral expression of shRNA, cDNA or mutants thereof will be utilized to evaluate putative individual driver loci, as well as combinatorial oncogene modules. This proposal directly addresses fundamental problems in the exploration and translation of novel colorectal cancer gene discovery in the context of clinical data which is available from TCGA.</p>

<p><strong>Stanford University<br />
Clinically-Relevant Regulatory Networks in the Lung Tumor Microenvironment<br />
PIs:  Sylvia K. Plevritis, Michael Clarke<br />
Project Abstract: <br /></strong>
Lung cancer is the leading cause of cancer death in the United States, accounting for approximately 160,000 deaths in 2009. The molecular mechanisms implicated in lung cancer development and progressions are not well understood. Recent evidence points to a complex interaction between the malignant cells and their microenvironment. However, much of our knowledge of the role of the tumor microenvironment comes from studies isolating the interactions between the malignant cells and a single component of the microenvironment, along a single pathway. We will reconstruct the first Tumor Microenvironment Interactome (TMI) of lung adenocarcinoma, which will identify global intra- and inter-cellular regulatory interactions between human malignant cells and their associated infiltrating immune cells, endothelial cells and fibroblasts. The TMI will be derived from global gene expression analysis of specific tumor microenvironment cell populations directly obtained from human lung cancer specimens using fluorescence-activated cell sorting (Specific Aim 1). The TMI will be reconstructed using novel computational approaches for inferring regulation among modules of genes (Specific Aim 2). From the TMI, we will identify candidate mediating factors, such as secreted cytokines, that regulate processes across the multiple cell subpopulations. We will specifically focus on the factors most associated with survival outcomes, by leveraging public domain expression data with long term survival outcomes (Specific Aim 2). We will use a combination of cell lines and animal models in the validation studies to test the effect of the candidate mediating factors on tumor behavior (Specific Aim 3). Through the reconstructed TMI, we will create a more global understanding of the lung tumor microenvironment. Our ultimate goal is to identify biologically and clinically relevant molecular targets that could be used to develop more effective therapies for lung cancer. The lung adenocarcinoma TMI will also be made publically available to the scientific research community as a hypothesis generation tool for evaluating the role of genes of interest.</p>

<p><strong>University of Connecticut School of Medicine/Dentistry<br />
Title:  Dynamics and Topology of Phosphotyrosine-SH2 Interaction Networks<br />
PIs:  Bruce J. Mayer, Forest M. White<br />
Project Abstract:<br /></strong>
Cell surface receptors linked to tyrosine kinases control a host of important cellular activities, including proliferation, differentiation, and motility. Disregulated tyrosine kinase signaling is a common feature of many human cancers, thus tyrosine kinases and their downstream effectors are targets for the development of new drugs for the treatment of cancer. In order to take full advantage of such promising new therapies, however, we need an understanding of how tyrosine kinase signaling networks process information on a systems level. While considerable progress has been made in developing quantitative models describing tyrosine kinase signaling networks, these efforts are severely hampered by a lack of quantitative information on how changes in tyrosine phosphorylation are coupled to their downstream effectors containing modular phosphotyrosine binding domains. The goals of this collaborative project are to take advantage of new experimental approaches to address this gap in knowledge directly. Specifically, we will use SH2 profiling, a phosphoproteomic approach that is highly complementary to mass spectrometry-based methods, to quantify dynamic changes in binding sites for specific effector proteins upon receptor tyrosine kinase activation. Responses to different receptors and in different cell types will be compared, allowing systems-level behavior to be correlated with biological outputs. We will also use single-molecule imaging methods to monitor the coupling of specific effectors to receptors in the intracellular environment. These studies will afford unprecedented insight into the interaction dynamics of receptor signaling complexes that will enable much more powerful and accurate models of tyrosine kinase signaling.</p>

<p><strong>University of Utah<br />
Integrative Signaling Models to Decipher Complex Cancer Phenotypes<br />
PIs:  Andrea H. Bild, Joe W. Gray, William E. Johnson, Laura J. Vanâ€™t Veer <br />
Project Abstract: <br /></strong>
The focus of our research is to investigate core signaling pathways that contribute to cancer growth, and to develop models to accurately determine optimal therapeutic regimens for cancer patients. Recent results from clinical trials using targeted therapies for solid tumors have shown that drug response is often times not driven by one mutation or pathway alone. Instead, response is confounded by interactions between the target gene and deregulation of downstream and alternative pathways. Therefore, our studies aim to model how signaling pathways work in relation to others in human tumors, and to identify patterns that correlate to drug response. We hypothesize that integrated 'omic' pathway models composed of multiple components of the growth factor receptor pathways will define biologically distinct subtypes of breast cancer and will accurately predict drug response in patient tumors. Specifically, we will develop and use genomic signatures centered on multiple levels of the growth factor receptor networks (GFRNs) to investigate how these pathways signal in human tumors. Novel statistical modeling approaches, including probabilistic barcode data standardization and Bayesian factor analysis for prediction of pathways and pathway interactions in tumors will move beyond individual pathway predictions to instead profile multi-pathway models in human tumors. Further, these models will integrate  'omic' data types, including RNA-sequencing, mutation status, and proteomic data, enabling a more comprehensive analysis of GFRN deregulation. GFRN pathway activity predictions and sensitivity/resistance to drugs that target the respective pathways will be validated in both cell lines as well as in "fresh" human tumor cells grown in 3-dimensional culture. Importantly, clinical validation of the pathway profiles will be carried out with I-SPY 2 (Investigation of Seril Studies to Predict Your Therapeutic Response with Imaging and Molecular Analysis 2) clinical trial data, which uses targeted therapies directed at GFR pathway components in the treatment of breast cancer. Ultimately, our studies will generate a series of well-validated pathway based biomarkers for individualized assessment of drug responsiveness, as well as interrogation of the coordinate deregulation of specific GFRN components in human tumors.</p>

<p><strong>Vanderbilt University<br />
Multiscale Modeling of ECM-tumor Interactions in Breast Cancer<br />
PIs:  Alissa Weaver, Yi Jiang, Patricia J. Keely<br />
Project Abstract: <br /></strong>
In humans, a common risk factor for the development of invasive breast cancer is dense breast tissue, detected by mammography. This dense tissue is associated with increased stromal collagen, increased epithelial cells, and decreased fatty tissue. Recent studies in transgenic mice indicate that increased collagen density in breast stromal tissue plays a causative role in promoting both the formation and invasiveness of breast tumors. Additional studies implicate tissue rigidity downstream of extracellular matrix (ECM) deposition and/or cross linking in promoting aggressive, invasive cellular phenotypes. Conversely, basement membrane ECM proteins that underlie normal and carcinoma in situ epithelial cells are thought to inhibit tumor progression. Because of the complexity of extracellular matrix-tumor cell interactions, it is difficult to fully isolate and understand the various effects of extracellular matrix on tumor progression using traditional biological approaches. We therefore propose to use an interdisciplinary approach in which we develop a 3-dimensional cell-based multiscale mathematical model of ECM-breast cancer interactions. Using this model, we will perform in silico experiments to test the overall hypothesis that ECM rigidity and stromal collagen fibrosis provide an environment that promotes tumor progression and invasion. We will specifically test the role of collagen fibril density, width, alignment, and crosslinking and the role of basement membrane ECM and proteases on cancer growth and invasion. All modeling will be fully integrated with experimentation to obtain realistic parameters and separately test predictions. We anticipate that this project will identify critical microenvironmental factors promoting breast cancer progression and lay the groundwork for future therapeutic intervention.</p>







            
        </div>
    
    

                                 </div>
    
                                 <div id="viewlet-below-content-body">



    <div class="visualClear"><!-- --></div>

    <div class="documentActions">
        

        

    </div>
</div>
    
                                
                            </div>
                        </td>
                        <td>
                            
                                
                            
                        </td>
                    </tr>
                    </table>
                    


                    
                </div>
            

            <div id="viewlet-below-content">





</div>
        </div>


    </div>


    <div class="row">
        <div class="subpage-footer">
           <div class="row">
<div class="cell width-full position-0">
        <div id="portal-footer"><div class="left"></div>
		<div class="footerLinks"><a href="../../contact_us.html" title="Contact Us">Contact Us</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www.nih.gov/about/privacy.htm" title="Privacy Notice">Privacy Notice</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www.nih.gov/about/disclaim.htm" title="Disclaimer">Disclaimer</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www3.cancer.gov/accessibility/nci508.htm" title="Accessibility">Accessibility</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://cancer.gov/global/viewing-files" target="_blank" title="Viewing Files">Viewing Files</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://ncicb.nci.nih.gov/support/" target="_blank" title="Application Support">Application Support</a></div>
       <div id="org"> 

			<div align="center">
<a href="http://www.dhhs.gov/"><img title="Department of Health and Human Services" src="../../footer_hhs.gif" width="45" height="43" alt="Department of Health and Human Services" border="0"></a>
<a href="http://www.nih.gov/"><img title="National Institutes of Health" src="../../footer_nih.gif" width="45" height="43" alt="National Institutes of Health" border="0"></a>
<a href="http://www.cancer.gov/"><img title="National Cancer Institute" src="../../footer_nci.gif" width="57" height="43" alt="National Cancer Institute" border="0"></a>
<a href="http://www.usa.gov/"><img title="USA.gov" src="../../footer_usagov.gif" width="93" height="43" alt="USA.gov" border="0"></a>
			</div>
<br /><br /><br />NIH...Turning Discovery Into Health

       </div>
        </div>



    </div>
</div>

        </div>
        <div id="kss-spinner">
            <img alt="" src="../../spinner.gif" />
        </div>
    </div>

</div>
</body>

<!-- Mirrored from icbp.nci.nih.gov/icbp-centers/grants/ by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 29 Sep 2016 19:35:44 GMT -->
</html>



