
<!DOCTYPE html PUBLIC
  "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" lang="en">

    
    
    
    
    



<!-- Mirrored from icbp.nci.nih.gov/resources/list-of-datasets-and-databases by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 29 Sep 2016 19:22:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />


    <link rel="stylesheet" type="text/css" href="includes/interactive_map.css" />
    <script src="../../ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
    <script type="text/javascript" src="includes/interactive_map.js"></script>

    
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />

    
        <base  /><!--[if lt IE 7]></base><![endif]-->
    

    
        
  
    <link rel="stylesheet" type="text/css" href="../portal_css/ICBP%20Theme/base-cachekey5221.css" />
        <!--[if lt IE 8]>    
    
    <link rel="stylesheet" type="text/css" media="screen" href="http://icbp.nci.nih.gov/portal_css/ICBP%20Theme/IEFixes-cachekey6323.css" />
        <![endif]-->
    
    <style type="text/css" media="screen">@import url(../portal_css/ICBP%20Theme/resourceplone.formwidget.autocompletejquery.autocomplete-cachekey0423.css);</style>
    <link rel="stylesheet" type="text/css" media="screen" href="../portal_css/ICBP%20Theme/resourceplone.app.dexterity.overlays-cachekey7584.css" />
    <style type="text/css" media="screen">@import url(../portal_css/ICBP%20Theme/resourcetheme.icbp.stylesheetsmain-cachekey7250.css);</style>
    <link rel="stylesheet" type="text/css" media="all" href="../portal_css/ICBP%20Theme/ploneCustom-cachekey8312.css" />

  
    <link rel="stylesheet" data-rel="kinetic-stylesheet" type="text/kss" href="../portal_kss/ICBP%20Theme/at-cachekey5785.css" />
  
    <script type="text/javascript" src="../portal_javascripts/ICBP%20Theme/jquery-cachekey2624.js"></script>
    <script type="text/javascript" src="../portal_javascripts/ICBP%20Theme/jquery-integration-cachekey3788.js"></script>
       <!--[if lt IE 8]>
     
    <script type="text/javascript" src="http://icbp.nci.nih.gov/portal_javascripts/ICBP%20Theme/iefixes-cachekey1480.js"></script>
       <![endif]-->
     
    <script type="text/javascript" src="../portal_javascripts/ICBP%20Theme/resourceplone.formwidget.autocompletejquery.autocomplete.min-cachekey0100.js"></script>





<link rel="kss-base-url" href="list-of-datasets-and-databases/index.html" />
<title>List of Datasets and Databases &mdash; Site</title>
        

    <link rel="shortcut icon" type="image/x-icon" href="../favicon.ico" />
    <link rel="apple-touch-icon" href="../touch_icon.png" />



    <link rel="home" href="../index.html" title="Front page" />

    <link rel="contents" href="../sitemap.html" title="Site Map" />






    <link rel="search" href="../search_form.html" title="Search this site" />



        
        
        
        
        

        <meta name="viewport" content="width=device-width; initial-scale=0.6666; maximum-scale=1.0; minimum-scale=0.6666" />
        <meta name="generator" content="Plone - http://plone.org" />
    
</head>

<body class="template-document_view portaltype-document site-icbp section-resources icons-on" dir="ltr">

<div id="visual-portal-wrapper">

<div id="logos">
   <div id="left"><img src="../nci.jpg" alt="" title="" height="43" width="286" /></div>
   <div id="right"><img src="../nih.jpg" alt="" title="" height="43" width="322" /></div>
</div>
 
        <div id="portal-top" class="row">
                <div class="cell width-full position-0">
                    <div id="portal-header">
    <p class="hiddenStructure">
  <a accesskey="2" href="list-of-datasets-and-databases.html#content">Skip to content.</a> |

  <a accesskey="6" href="list-of-datasets-and-databases.html#portal-globalnav">Skip to navigation</a>
</p>

<div id="portal-personaltools-wrapper">

<h5 class="hiddenStructure">Personal tools</h5>





</div>



<div id="portal-searchbox">
    <form name="searchform" id="searchGadget_form" action="http://icbp.nci.nih.gov/search">

        <div class="LSBox">
        <label class="hiddenStructure" for="searchGadget">Search Site</label>

        <input name="SearchableText" type="text" size="18" title="Search Site" accesskey="4" class="searchField inputLabel" id="searchGadget" />

        <input class="searchButton" type="submit" value="Search" />

        <div class="searchSection">
            <input id="searchbox_currentfolder_only" class="noborder" type="checkbox" name="path" value="/icbp/resources" />
            <label for="searchbox_currentfolder_only" style="cursor: pointer">
                only in current section
            </label>
        </div>

        <div class="LSResult" id="LSResult" style=""><div class="LSShadow" id="LSShadow"></div></div>
        </div>
    </form>

    <div id="portal-advanced-search" class="hiddenStructure">
        <a href="../search_form.html" accesskey="5">
            Advanced Search&hellip;
        </a>
    </div>

</div>
<a id="portal-logo" title="Site" accesskey="1" href="../index.html">
    <img src="../logo.png" alt="Site" title="Site" height="100" width="500" /></a>

    <h5 class="hiddenStructure">Sections</h5>

    <ul id="portal-globalnav">
<li id="portaltab-index_html" class="plain"><div class="left"></div><a href="../index.html" title="">Home</a><div class="right" /></li>
<li id="portaltab-about" class="plain"><div class="left"></div><a href="../about.html" title="">About ICBP</a><div class="right" /></li>
<li id="portaltab-icbp-centers" class="plain"><div class="left"></div><a href="../icbp-centers.html" title="">ICBP Centers</a><div class="right" /></li>
<li id="portaltab-resources" class="selected"><div class="left"></div><a href="../resources.html" title="">Resources</a><div class="right" /></li>
<li id="portaltab-icbp-news" class="plain"><div class="left"></div><a href="../icbp-news.html" title="">ICBP News</a><div class="right" /></li>
<li id="portaltab-education-and-outreach" class="plain"><div class="left"></div><a href="../education-and-outreach.html" title="">Education and Outreach</a><div class="right" /></li>

</ul>

</div>

                </div>
        </div>

        
        


        
    <div id="portal-columns" class="row">

        <div id="portal-column-content" class="cell width-full position-0">

            <div id="viewlet-above-content"><div id="portal-breadcrumbs">

    <span id="breadcrumbs-you-are-here">You
are here:</span>
    <span id="breadcrumbs-home">
        <a href="../index.html">Home</a>
        <span class="breadcrumbSeparator">
            &rsaquo;
            
        </span>
    </span>
    <span id="breadcrumbs-1" dir="ltr">
        
            <a href="../resources.html">Resources</a>
            <span class="breadcrumbSeparator">
                &rsaquo;
                
            </span>
            
         
    </span>
    <span id="breadcrumbs-2" dir="ltr">
        
            
            
            <span id="breadcrumbs-current">List of Datasets and Databases</span>
         
    </span>

</div>
</div>

            
                <div class="">

                    

                    

    <dl class="portalMessage info" id="kssPortalMessage" style="display:none">
        <dt>Info</dt>
        <dd></dd>
    </dl>



                    
                    <table cellpadding="0" cellspacing="0" border="0" id="contentTable">
                    <tr>
                        <td valign="top">
                            <div id="content" style="">
                                
    
                                
    
                                 <div id="viewlet-above-content-title"></div>
                                 
                                     
                                     
            
                <h1 id="parent-fieldname-title" class="documentFirstHeading">
                    List of Datasets and Databases
                </h1>
            
            
        
                                 
                                 <div id="viewlet-below-content-title">
</div>
    
                                 
                                     
                                     
            
                
            
            
        
                                 
    
                                 <div id="viewlet-above-content-body">

</div>
                                 <div id="content-core">
                                     
    
        
        <div class="" id="parent-fieldname-text-3d7bb7eb-6614-478f-ba23-0153cdf4b759">
            <p><strong>The Broad Institute/Dana Farber Cancer Institute<br /> Principal Investigator: Todd Golub, M.D. <br /></strong></p>
<p>Databases:</p>
<p>Cancer Program Datasets Portal</p>
<p>PI: Todd Golub, M.D.<br /> Contact: Michael Reich (<a href="mailto:michaelr@broadinstitute.org" title="michaelr@broadinstitute.org">michaelr@broadinstitute.org</a>)<br /> Web Link: <a href="http://www.broadinstitute.org/cgi-bin/cancer/datasets.cgi" target="_blank" title="http://www.broadinstitute.org/cgi-bin/cancer/datasets.cgi">http://www.broadinstitute.org/cgi-bin/cancer/datasets.cgi</a></p>
<p>Collection of over 100 datasets from Broad Institute Cancer Program publications, providing gene expression, and SNP microarray assays and RNAi screens of a number of cancer types including breast, colorectal, hepatocellular, leukemia, lung, lymphoma, medulloblastoma, melanoma, nasopharyngeal, neuroblastoma, prostate, and sarcoma.</p>
<p><strong>Molecular Signature Database (MSigDB)</strong><br /> PI: Jill P. Mesirov, Ph.D.<br /> Contact: <a href="mailto:gsea@broadinstitute.org" title="gsea@broadinstitute.org">gsea@broadinstitute.org</a><br /> Web Link: <a href="http://www.broadinstitute.org/gsea/msigdb/" target="_blank" title="http://www.broadinstitute.org/gsea/msigdb/">http://www.broadinstitute.org/gsea/msigdb/</a></p>
<p>The Molecular Signatures Database (MSigDB) is a collection of gene sets for use with GSEA software. From this web site, you can search for gene sets, browse gene sets, view pathway annotations, download gene sets, compute overlaps between gene sets, categorize members of a gene set by gene families, and build an expression signature of a gene set using a compendium of expression profiles.</p>
<p><strong>Datasets:</strong></p>
<p><strong>Achilles Project dataset 1.0 (102 cell lines)</strong><br /> PI: William Hahn, M.D., Ph.D.<br /> Contact: Paquita Vazquez (<a href="mailto:Vazquez@broadinstitute.org" title="Vazquez@broadinstitute.org">Vazquez@broadinstitute.org</a>)<br /> Web Link: <a href="http://www.broadinstitute.org/icbp/home" target="_blank" title="http://www.broadinstitute.org/icbp/home">http://www.broadinstitute.org/icbp/home</a></p>
<p>shRNA level data from highly parallel pooled genome-wide shRNA screens across 102 cell lines.</p>
<hr />
<p></p>
<p><strong>Columbia University<br /> Principal Investigator: Andrea Califano, Ph.D.</strong></p>
<p><strong>B Cell Interactome</strong><br /> Contact: Celine Lefebvre, Ph.D., Columbia University<br /> Web Link: <a href="http://amdec-bioinfo.cu-genome.org/html/BCellInteractome.html" target="_blank" title="http://amdec-bioinfo.cu-genome.org/html/BCellInteractome.html">http://amdec-bioinfo.cu-genome.org/html/BCellInteractome.html</a></p>
<p>The B cell interactome (BCI) is a network of protein-protein, protein-DNA and modulatory interactions in human B cells. The network contains known interactions (reported in public databases) and predicted interactions by a Bayesian evidence integration framework which integrates a variety of generic and context specific experimental clues about protein-protein and protein-DNA interactions - such as a large collection of B cell expression profiles - with inferences from different reverse engineering algorithms, such as GeneWays and ARACNE. Modulatory interactions are predicted by MINDY, an algorithm for the prediction of modulators of transcriptional interactions.</p>
<p><strong>TransfactomeDB</strong><br /> Contact: Harmen Bussemaker, Ph.D., Columbia University<br /> Web Link: <a href="http://bussemaker.bio.columbia.edu:8080/YeastTransfactomeDB/" target="_blank" title="http://bussemaker.bio.columbia.edu:8080/YeastTransfactomeDB/">http://bussemaker.bio.columbia.edu:8080/YeastTransfactomeDB/</a></p>
<p>Accurate and comprehensive information about the nucleotide sequence specificity of trans-acting factors (TFs) is essential for computational and experimental analyses of gene regulatory networks. The Yeast Transfactome Database is a repository of sequence specificity models and condition-specific regulatory activities for a large number of DNA- and RNA-binding proteins in Saccharomyces cerevisiae. The sequence specificities in TransfactomeDB, represented as position-specific affinity matrices (PSAMs), are directly estimated from genomewide measurements of TF-binding using our previously published MatrixREDUCE algorithm, which is based on a biophysical model. For each mRNA expression profile in the NCBI Gene Expression Omnibus, we used sequence-based regression analysis to estimate the post-translational regulatory activity of each TF for which a PSAM is available. The trans-factor activity profiles across multiple experiments available in TransfactomeDB allow the user to explore potential regulatory roles of hundreds of TFs in any of thousands of microarray experiments.</p>
<p><strong>PREPPI</strong><br /> Contact: Barry Honig, PhD, Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://bhapp.c2b2.columbia.edu/PREPPI/" target="_blank" title="http://bhapp.c2b2.columbia.edu/PREPPI/">http://bhapp.c2b2.columbia.edu/PREPPI/</a></p>
<p>PREPPI is a database of predicted and experimentally determined protein-protein interactions (PPIs) for yeast and human. Predicted interactions are assigned a likelihood using a Bayesian framework that combines structural, functional, evolutionary and expression information. The database contains ~2 million predictions including 31,402 for yeast and 317,813 for human that are considered high confidence based on our analysis. Experimentally determined interactions are compiled from a set of public databases (e.g., DIP, IntAct, HPRD, etc).</p>
<hr />
<p></p>
<p><strong>Georgetown University</strong><br /> Principal Investigator: Robert Clarke, Ph.D.</p>
<p><strong>Databases:</strong></p>
<p><strong>Georgetown Database of Cancer (G-DOC)</strong><br /> Contact: Subha Madhavan (<a href="mailto:sm696@georgetown.edu" title="sm696@georgetown.edu">sm696@georgetown.edu</a>)<br /> <a href="https://gdoc.georgetown.edu/gdoc/" target="_blank" title="https://gdoc.georgetown.edu/gdoc/">https://gdoc.georgetown.edu/gdoc/</a></p>
<p>G-DOC is a cutting edge data integration platform and knowledge discovery system for the oncology and translational research communities, storing both omics and clinical data from several studies along with analysis tools.</p>
<p><strong>Datasets:</strong></p>
<p><strong>GEO: GSE8562</strong><br /> Contact: Rebecca Riggins, Ph.D. (<a href="mailto:rbr7@georgetown.edu" title="rbr7@georgetown.edu">rbr7@georgetown.edu</a>) <br /> Web Link: <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE8562" target="_blank" title="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE8562">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE8562</a></p>
<p>XBP1 confers estrogen independence and anti-estrogen resistance in breast cancer cells.</p>
<p>Contact: Yuriy Gusev, Ph.D. (<a href="mailto:yg63@georgetown.edu" title="yg63@georgetown.edu">yg63@georgetown.edu</a></p>
<p>)</p>
<p>The full collection of uniformly processed public “omics” data provided on the NCI’s SFTP site (sftp://caftps.nci.nih.gov) can be seen in the table below. The original, unprocessed raw data sets can be found at GEO using the appropriate identifier in the table. This data collection includes gene expression, copy number, micro RNA, and other types of “omics” data, along with (in most cases) clinical information.</p>
<p><strong>Multi-omics data for breast cancer cell lines to understand tamoxifen resistance. </strong><br /> Contact: Ayesha Shajahan, Ph.D.</p>
<p>This data collection includes gene expression, copy number, micro RNA and metabolomics data, along with clinical information on breast cancer cell lines. (will be made available upon publication)</p>
<hr />
<p></p>
<p><strong>Massachusetts Institute of Technology (MIT)</strong><br /> Principal Investigator: Doug Lauffenburger, Ph.D.</p>
<p><strong>The MIT/ICBP siRNA Database</strong><br /> Contact: Mary Lindstrom, Ph.D. (<a href="mailto:mwernett@mit.edu" title="mwernett@mit.edu">mwernett@mit.edu</a>)<br /> Web Link: <a href="http://web.mit.edu/sirna" target="_blank" title="http://web.mit.edu/sirna">http://web.mit.edu/sirna</a></p>
<p>An on-line database of experimentally validated siRNAs and shRNAs against target genes, with a focus on genes thought to be involved in cancer.</p>
<p><strong>MIT ICBP Website</strong><br /> Contact: Brian Joughin, Ph.D. (<a href="mailto:bjoughin@mit.edu" title="bjoughin@mit.edu">bjoughin@mit.edu</a>) Web Link: <a href="http://web.mit.edu/icbp/data" target="_blank" title="http://web.mit.edu/icbp/data">http://web.mit.edu/icbp/data</a></p>
<p>All MIT ICBP publications are listed as they are published. Links are provided to abstracts at PubMed, and where appropriate, supplemental data files are hosted as well.</p>
<p><strong>PTMScout</strong><br /> Contact: <a href="mailto:ptmscout_admin@mit.edu" title="ptmscout_admin@mit.edu">ptmscout_admin@mit.edu</a><br /> Web Link: <a href="http://ptmscout.mit.edu/" target="_blank" title="http://ptmscout.mit.edu">http://ptmscout.mit.edu</a></p>
<p>PTMScout is a tool for viewing and analyzing high-throughput post-translational modification proteomic data, with a particular eye towards novel hypothesis generation. External data from Gene Ontology, PFAM, and Scansite are integrated with user data automatically, as well as expression data in a large number of tissue types and cell lines. Additionally, all post-translational proteomic mass spectrometry datasets from the White Lab at MIT are banked here, and other users may include their data on request.</p>
<hr />
<p></p>
<p><strong>Memorial Sloan-Kettering Cancer Center<br /> Principal Investigator: Chris Sander, Ph.D.</strong></p>
<p><strong>Pathway Commons</strong><br /> Contact: <a href="mailto:pc-info@pathwaycommons.org" title="pc-info@pathwaycommons.org">pc-info@pathwaycommons.org</a><br /> Web Link: <a href="http://www.pathwaycommons.org/" target="_blank" title="http://www.pathwaycommons.org">http://www.pathwaycommons.org</a></p>
<p>Pathway Commons is a convenient point of access to biological pathway information collected from public pathway databases, which you can browse or search. Pathways include biochemical reactions, complex assembly, transport and catalysis events, and physical interactions involving proteins, DNA, RNA, small molecules and complexes.</p>
<p><strong>cBio Cancer Genomics Portal</strong><br /> Contact: <a href="mailto:cbioportal@googlegroups.com" title="cbioportal@googlegroups.com">cbioportal@googlegroups.com</a><br /> Web Link: <a href="http://www.cbioportal.org/" target="_blank" title="http://www.cbioportal.org/">http://www.cbioportal.org/</a></p>
<p>The cBio Cancer Genomics Portal provides direct download and visualization of large-scale cancer genomics data sets. It currently provides direct access to over 5,000 tumor samples in 20 cancer types. Currently, the cBio Cancer Genomics Portal contains data sets from The Cancer Genome Atlas (TCGA) project and from MSKCC-funded cancer genome sequencing projects (prostate and sarcoma). Cancer genomics datasets from additional tumor types and cancer cell line profiling will be added as the data becomes available.</p>
<hr />
<p></p>
<p><strong>Houston Methodist Research Institute<br /> Principal Investigator: Stephen Wong, Ph.D.</strong></p>
<p>None.</p>
<hr />
<p></p>
<p><strong>Oregon Health &amp; Science University<br /> Principal Investigator: Joe Gray, Ph.D.</strong></p>
<p>Contact: Laura Heiser, Ph.D. (<a href="mailto:heiserl@ohsu.edu" title="heiserl@ohsu.edu">heiserl@ohsu.edu</a>)</p>
<p>The OHSU ICBP has generated a substantial profiling compendium on a panel of ~50 breast cancer cell lines. Cell lines in the collection were selected to mirror the phenotypic, genomic, and expression heterogeneity of human breast cancers as well as important recurrent molecular features found in primary tumors. The lines were characterized in extensive molecular detail in order to allow identification of molecular features most strongly associated with biological response. Molecular features measured include genome copy number (SNP6.0 arrays), gene expression (U133A, Exon Array, RNAseq), alternative splicing (exon array, RNAseq), mutation detection (Exome seq), miRNA expression, gene promoter methylation (Illumina Infinium Kit), cytokine secretion, protein expression (Western blots and RPPA arrays), and drug response (&gt;100 compounds). Additionally, we have conducted two detailed studies of the dynamics of drug response, in which a select set of cell lines were treated with up to 4 therapeutic compounds, and then profiled at 8 time-points to assess changes in protein expression, as measured by RPPA array.</p>
<hr />
<p></p>
<p><strong>Sage Bionetworks<br /> Principal Investigator: Stephen Friend, M.D., Ph.D.</strong></p>
<p><strong>SYNAPSE</strong><br /> Conatct: <a href="mailto:synapseInfo@sagebase.org" title="synapseInfo@sagebase.org">synapseInfo@sagebase.org</a><br /> Web Link: <a href="https://synapse.sagebase.org/ - Home:null" target="_blank" title="https://synapse.sagebase.org/ - Home:null">https://synapse.sagebase.org/ - Home:null</a></p>
<p>Synapse is an innovation compute space that brings together scientific data, tools, and disease models into a Commons that enables true collaborative research for those who want to share and analyze data together. The platform consists of a web portal, web services, and integrations with data analysis tool and is organized around novel “Analysis Communities” that any scientist can create or join to work on complex medical problems. In Synapse you can:</p>
<ul>
<li>Search across public data including curated GEO, TCGA, and Array Express</li>
<li>Find raw data or follow an analysis path to processed results</li>
<li>Load data directly into analytical tools like R, and record your analysis</li>
<li>Create Public or Private projects to share your work</li>
</ul>
<p>Synapse is designed to enable many layers of collaboration: within a single team, between research teams, between research teams and industry partners, and within pharmaceutical companies.</p>
<p>Examples of datasets in Synapse:</p>
<p><strong>Synapse Commons Repository</strong><br /> Conatct: <a href="mailto:synapseInfo@sagebase.org" title="synapseInfo@sagebase.org">synapseInfo@sagebase.org</a><br /> Web Link: <a href="https://synapse.sagebase.org/ - Synapse:syn150935" target="_blank" title="https://synapse.sagebase.org/ - Synapse:syn150935">https://synapse.sagebase.org/ - Synapse:syn150935</a></p>
<p>The Synapse Commons Repository is a convenient repository of raw genomic data and corresponding phenotypic information from publicly available datasets, available with no restrictions on use. The data are from public repositories like NCBI GEO, ArrayExpress from EBI, or TCGA, as well as any source that can be accessed through the web, including a URL made available through an individual lab, a core facility, or a cloud based storage site like an Amazon S3 bucket.</p>
<p><strong>Sage / DREAM breast cancer prognosis challenge</strong><br /> Web Link: <a href="https://synapse.sagebase.org/#BCCOverview:0" target="_blank" title="https://synapse.sagebase.org/#BCCOverview:0">https://synapse.sagebase.org/#BCCOverview:0</a></p>
<p>The Sage / DREAM Breast Cancer Prognosis Challenge, announced in April 2012 and concluding at the end of this year, is intended to assess the accuracy of predictive computational models designed to predict breast cancer survival. The computational models are built using both the clinical information about the patient's tumor as well as genome-wide molecular profiling data including gene expression and copy number profiles. The Challenge is fueled by the generous donations of two breast cancer clinical study data sets (Metabric and Oslo Val) obtained by Carlos Caldas of Cancer Research UK and Anne-Lise Borresen-Dale of Oslo University Hospital.</p>
<hr />
<p></p>
<p><strong>St. Elizabeth’s Medical Center/Tufts University<br /> Principal Investigator: Lynn Hlatky, Ph.D.</strong></p>
<p><strong>Tumor Dormancy Gene Expression Profiling</strong><br /> Contact: Edward Rietman, Ph.D. (<a href="mailto:Edward.Rietman@tufts.edu" title="Edward.Rietman@tufts.edu">Edward.Rietman@tufts.edu</a>)<br /> Web Link: <a href="http://www.ebi.ac.uk/microarray-as/ae/browse.html?keywords=E-TABM-390&amp;species=&amp;array=&amp;exptype%5B%5D=&amp;exptype%5B%5D=&amp;pagesize=25&amp;sortby=releasedate&amp;sortorder=descending" target="_blank" title="http://www.ebi.ac.uk/microarray-as/ae/browse.html?keywords=E-TABM-390&species=&array=&exptype%5B%5D=&exptype%5B%5D=&pagesize=25&sortby=releasedate&sortorder=descending">http://www.ebi.ac.uk/microarray-as/ae/browse.html?keywords=E-TABM-390&amp;species=&amp;array=&amp;exptype%5B%5D=&amp;exptype%5B%5D=&amp;pagesize=25&amp;sortby=releasedate&amp;sortorder=descending</a></p>
<p>Whole genome expression profiling (Agilent) of tumor tissue from experiments directed toward tumor dormancy and emergence from dormancy.</p>
<p><strong>Gene Expression Data for Low Dose Radiation Experiments</strong><br /> Contact: Edward Rietman, Ph.D. (<a href="mailto:Edward.Rietman@tufts.edu" target="_blank" title="Edward.Rietman@tufts.edu">Edward.Rietman@tufts.edu</a>)<br /> Web Link: not yet active</p>
<p>Experiments included primary normal human prostate fibroblast and epithelial cell cultures from several individuals, normal human dermal fibroblasts, endothelial cells, mesenchymal stem cells and lymphocytes purified from several normal volunteers peripheral blood. Exposures ranged from acute exposure at 1 to 200 cGY to long-term chronic exposures at dose-rate of less than 2.45microGy per minute and time-points for chronic exposures extending over 2 weeks.</p>
<p><strong>Prostate Cancer Clinical Data</strong><br /> Contact: Edward Rietman, Ph.D. (<a href="mailto:Edward.Rietman@tufts.edu" title="Edward.Rietman@tufts.edu">Edward.Rietman@tufts.edu</a>)<br /> Web Link: not yet active</p>
<p>Data samples from 700+ clinical specimens of different stages collected at Steward St. Elizabeth’s Medical Center.</p>
<hr />
<p></p>
<p><strong>Stanford University School of Medicine<br /> Principal Investigator: Sylvia Plevritis, Ph.D.</strong></p>
<p><strong>Cytobank</strong><br /> Contact: Help desk (<a href="mailto:info@cytobank.org" title="info@cytobank.org">info@cytobank.org</a>)<br /> Web Link: <a href="http://www.cytobank.org/" target="_blank" title="http://www.cytobank.org/">http://www.cytobank.org/</a></p>
<p>Manage, analyze and share flow cytometry data over the web.</p>
<hr />
<p></p>
<p><strong>University of Texas Health Science Center, San Antonio<br /> Principal Investigator: Tim H-M Huang, Ph.D.</strong></p>
<p><strong>Kmethylome</strong><br /> Contact: Yidong Chen, Ph.D. (<a href="mailto:chenY8@uthscsa.edu" title="chenY8@uthscsa.edu">chenY8@uthscsa.edu</a>) and Mark Doderer, Ph.D. (<a href="mailto:Doderer@uthscsa.edu" title="Doderer@uthscsa.edu">Doderer@uthscsa.edu</a>)<br /> Web Link: <a href="http://cbbiweb.uthscsa.edu/KMethylomes/" target="_blank" title="http://cbbiweb.uthscsa.edu/KMethylomes/">http://cbbiweb.uthscsa.edu/KMethylomes/</a></p>
<p>A comprehensive and interactive viewer for genome-wide methylation of tumors and normal samples, enabling close examination within and around a gene, visualization of custom-tracks and performing differential methylation analysis. Additionally, the framework provides a gene centric view of methylation data that allows for visualization of methylation similarities across tumors between gene sets including pathway membership, similar functional annotation, and correlated methylation values.</p>
<p><strong>Estrogen Receptor Target Database (ERTargetDB)</strong><br /> Contact: Victor Jin, Ph.D. (<a href="mailto:Victor.Jin@osumc.edu" title="Victor.Jin@osumc.edu)">Victor.Jin@osumc.edu)</a>)<br /> Web Link: <a href="http://bioinformatics.wistar.upenn.edu/ERTargetDB/" target="_blank" title="http://bioinformatics.wistar.upenn.edu/ERTargetDB/">http://bioinformatics.wistar.upenn.edu/ERTargetDB/</a></p>
<p>ERTargetDB integrates information from the ongoing Chip-on-chip experiments in our laboratory and promoter sequence conservation from the OMGProm database. We will add methylation and acetylation patterns across the target gene promoters in the future versions of the database.</p>
<p><strong>Hormone Receptor Target Binding Loci Database (HRTBLDB)</strong><br /> Contact: Victor Jin, Ph.D. (<a href="mailto:Victor.Jin@osumc.edu" title="Victor.Jin@osumc.edu">Victor.Jin@osumc.edu</a>)<br /> Web Link: <a href="http://motif.bmi.ohio-state.edu/hrtbldb/" target="_blank" title="http://motif.bmi.ohio-state.edu/hrtbldb/">http://motif.bmi.ohio-state.edu/hrtbldb/</a></p>
<p>Data from high-throughput lab techniques (ChIP-chip, ChIP-Seq, and ChIP-PET) are integrated in to this unified database to aid research on the hormone signaling pathways that regulate gene expression. Hormone induced differential gene expression has been implicated in numerous pathological conditions. Given this implication, research in hormone receptor binding sites is increasing at a rapid pace, and with the advent of whole genome analysis, the volume of data generated by this research is immense. HRTBLDb is built to help researchers collaborate and prosper from this explosion of data.</p>
<p><strong>Endometrial Cancer Patient Methylation Database (ECPMDb)</strong><br /> Contact: Victor Jin, Ph.D. (<a href="mailto:Victor.Jin@osumc.edu" title="Victor.Jin@osumc.edu">Victor.Jin@osumc.edu</a>)<br /> Web Link: <a href="http://motif.bmi.ohio-state.edu/brcaChileV2/" target="_blank" title="http://motif.bmi.ohio-state.edu/brcaChileV2/">http://motif.bmi.ohio-state.edu/brcaChileV2/</a></p>
<p>The Endometrial Cancer Patient Methylation Database (ECPMDb) consists of whole genome methylation data in 92 endometrial cancer patients and 12 normal tissues conducted by the MIRA-seq protocol. The ECPMDb makes rich, interactive genomic visualizations of the hyper-methylated regions across the human genome and can be accessible over the internet and be shared with collaborators. The graphical rendering of the data is done in a dynamic, modular fashion, using Scalable Vector Graphics that are rendered in the browser by Javascript. The SVG renderer is run by a custom Javascript program using the jQuery and Raphael Javascript libraries. The data is supplied to the client in Javascript Object Notation from the server, which is then used by the program to dynamically render both the SVG graphical visualization and the HTML tabular representation of the data being visualized. The primary advantage of this database visualization is its rich and interactive data display. Moving the cursor over an element in the browser will display its coordinates, the GC content at that site, or other information as assigned.</p>
<p><strong>mirPromoter</strong><br /> Contact: Yunlong Liu, Ph.D.<br /> Web Link: <a href="http://compbio.iupui.edu/group/6/pages/mirpromoter" target="_blank" title="http://compbio.iupui.edu/group/6/pages/mirpromoter">http://compbio.iupui.edu/group/6/pages/mirpromoter</a></p>
<p>ChIP-seq data, microarray data, and microRNA promoter prediction data.</p>
<p><strong>miR2Disease</strong><br /> Contact: Yunlong Liu, Ph.D. (<a href="mailto:yunliu@iupui.edu" title="yunliu@iupui.edu">yunliu@iupui.edu</a>)<br /> Web Link: <a href="http://www.mir2disease.org/" target="_blank" title="http://www.mir2disease.org/">http://www.mir2disease.org/</a></p>
<p>miR2Disease, a manually curated database, aims at providing a comprehensive resource of miRNA deregulation in various human diseases. Each entry in the miR2Disease contains detailed information on a miRNA-disease relationship, including miRNA ID, disease name, a brief description of the miRNA-disease relationship, miRNA expression pattern in the disease state, detection method for miRNA expression, experimentally verified miRNA target gene(s), and literature reference.</p>
<hr />
<p></p>
<p><strong>Vanderbilt University Medical Center<br /> Principal Investigator: Vito Quaranta, M.D.</strong></p>
<p><strong>Datasets</strong></p>
<p><strong>CCSB@V Data</strong><br /> Web Link: <a href="http://ccsb.vanderbilt.edu/ccsb/research/datasets" target="_blank" title="http://ccsb.vanderbilt.edu/ccsb/research/datasets">http://ccsb.vanderbilt.edu/ccsb/research/datasets</a><br /> Contact: Shawn Garbett (<a href="mailto:shawn.garbett@vanderbilt.edu" title="shawn.garbett@vanderbilt.edu">shawn.garbett@vanderbilt.edu</a>)</p>
<p>This is a listing of the different datasets that have been generated in our Center. Solely informative, the data itself is not yet directly available. The names of the experimentalists who generated the data are associated with each dataset and can be contacted for more information.</p>
<p><strong>ICBP Portal</strong><br /> Web Link: <a href="http://www.broadinstitute.org/icbp_profiles" target="_blank" title="http://www.broadinstitute.org/icbp_profiles">http://www.broadinstitute.org/icbp_profiles</a></p>
<p>We have uploaded various datasets in the ICBP Portal including cell motility and drug response live imaging movies.</p>
            
        </div>
    
    

                                 </div>
    
                                 <div id="viewlet-below-content-body">



    <div class="visualClear"><!-- --></div>

    <div class="documentActions">
        

        

    </div>
</div>
    
                                
                            </div>
                        </td>
                        <td>
                            
                                
                            
                        </td>
                    </tr>
                    </table>
                    


                    
                </div>
            

            <div id="viewlet-below-content">





</div>
        </div>


    </div>


    <div class="row">
        <div class="subpage-footer">
           <div class="row">
<div class="cell width-full position-0">
        <div id="portal-footer"><div class="left"></div>
		<div class="footerLinks"><a href="../contact_us.html" title="Contact Us">Contact Us</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www.nih.gov/about/privacy.htm" title="Privacy Notice">Privacy Notice</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www.nih.gov/about/disclaim.htm" title="Disclaimer">Disclaimer</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www3.cancer.gov/accessibility/nci508.htm" title="Accessibility">Accessibility</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://cancer.gov/global/viewing-files" target="_blank" title="Viewing Files">Viewing Files</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://ncicb.nci.nih.gov/support/" target="_blank" title="Application Support">Application Support</a></div>
       <div id="org"> 

			<div align="center">
<a href="http://www.dhhs.gov/"><img title="Department of Health and Human Services" src="../footer_hhs.gif" width="45" height="43" alt="Department of Health and Human Services" border="0"></a>
<a href="http://www.nih.gov/"><img title="National Institutes of Health" src="../footer_nih.gif" width="45" height="43" alt="National Institutes of Health" border="0"></a>
<a href="http://www.cancer.gov/"><img title="National Cancer Institute" src="../footer_nci.gif" width="57" height="43" alt="National Cancer Institute" border="0"></a>
<a href="http://www.usa.gov/"><img title="USA.gov" src="../footer_usagov.gif" width="93" height="43" alt="USA.gov" border="0"></a>
			</div>
<br /><br /><br />NIH...Turning Discovery Into Health

       </div>
        </div>



    </div>
</div>

        </div>
        <div id="kss-spinner">
            <img alt="" src="../spinner.gif" />
        </div>
    </div>

</div>
</body>

<!-- Mirrored from icbp.nci.nih.gov/resources/list-of-datasets-and-databases by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 29 Sep 2016 19:22:21 GMT -->
</html>



