
<!DOCTYPE html PUBLIC
  "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" lang="en">

    
    
    
    
    



<!-- Mirrored from icbp.nci.nih.gov/resources/software by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 29 Sep 2016 19:22:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />


    <link rel="stylesheet" type="text/css" href="includes/interactive_map.css" />
    <script src="../../ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
    <script type="text/javascript" src="includes/interactive_map.js"></script>

    
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />

    
        <base  /><!--[if lt IE 7]></base><![endif]-->
    

    
        
  
    <link rel="stylesheet" type="text/css" href="../portal_css/ICBP%20Theme/base-cachekey5221.css" />
        <!--[if lt IE 8]>    
    
    <link rel="stylesheet" type="text/css" media="screen" href="http://icbp.nci.nih.gov/portal_css/ICBP%20Theme/IEFixes-cachekey6323.css" />
        <![endif]-->
    
    <style type="text/css" media="screen">@import url(../portal_css/ICBP%20Theme/resourceplone.formwidget.autocompletejquery.autocomplete-cachekey0423.css);</style>
    <link rel="stylesheet" type="text/css" media="screen" href="../portal_css/ICBP%20Theme/resourceplone.app.dexterity.overlays-cachekey7584.css" />
    <style type="text/css" media="screen">@import url(../portal_css/ICBP%20Theme/resourcetheme.icbp.stylesheetsmain-cachekey7250.css);</style>
    <link rel="stylesheet" type="text/css" media="all" href="../portal_css/ICBP%20Theme/ploneCustom-cachekey8312.css" />

  
    <link rel="stylesheet" data-rel="kinetic-stylesheet" type="text/kss" href="../portal_kss/ICBP%20Theme/at-cachekey5785.css" />
  
    <script type="text/javascript" src="../portal_javascripts/ICBP%20Theme/jquery-cachekey2624.js"></script>
    <script type="text/javascript" src="../portal_javascripts/ICBP%20Theme/jquery-integration-cachekey3788.js"></script>
       <!--[if lt IE 8]>
     
    <script type="text/javascript" src="http://icbp.nci.nih.gov/portal_javascripts/ICBP%20Theme/iefixes-cachekey1480.js"></script>
       <![endif]-->
     
    <script type="text/javascript" src="../portal_javascripts/ICBP%20Theme/resourceplone.formwidget.autocompletejquery.autocomplete.min-cachekey0100.js"></script>





<link rel="kss-base-url" href="software/index.html" />
<title>Software &mdash; Site</title>
        

    <link rel="shortcut icon" type="image/x-icon" href="../favicon.ico" />
    <link rel="apple-touch-icon" href="../touch_icon.png" />



    <link rel="home" href="../index.html" title="Front page" />

    <link rel="contents" href="../sitemap.html" title="Site Map" />






    <link rel="search" href="../search_form.html" title="Search this site" />



        
        
        
        
        

        <meta name="viewport" content="width=device-width; initial-scale=0.6666; maximum-scale=1.0; minimum-scale=0.6666" />
        <meta name="generator" content="Plone - http://plone.org" />
    
</head>

<body class="template-document_view portaltype-document site-icbp section-resources icons-on" dir="ltr">

<div id="visual-portal-wrapper">

<div id="logos">
   <div id="left"><img src="../nci.jpg" alt="" title="" height="43" width="286" /></div>
   <div id="right"><img src="../nih.jpg" alt="" title="" height="43" width="322" /></div>
</div>
 
        <div id="portal-top" class="row">
                <div class="cell width-full position-0">
                    <div id="portal-header">
    <p class="hiddenStructure">
  <a accesskey="2" href="software.html#content">Skip to content.</a> |

  <a accesskey="6" href="software.html#portal-globalnav">Skip to navigation</a>
</p>

<div id="portal-personaltools-wrapper">

<h5 class="hiddenStructure">Personal tools</h5>





</div>



<div id="portal-searchbox">
    <form name="searchform" id="searchGadget_form" action="http://icbp.nci.nih.gov/search">

        <div class="LSBox">
        <label class="hiddenStructure" for="searchGadget">Search Site</label>

        <input name="SearchableText" type="text" size="18" title="Search Site" accesskey="4" class="searchField inputLabel" id="searchGadget" />

        <input class="searchButton" type="submit" value="Search" />

        <div class="searchSection">
            <input id="searchbox_currentfolder_only" class="noborder" type="checkbox" name="path" value="/icbp/resources" />
            <label for="searchbox_currentfolder_only" style="cursor: pointer">
                only in current section
            </label>
        </div>

        <div class="LSResult" id="LSResult" style=""><div class="LSShadow" id="LSShadow"></div></div>
        </div>
    </form>

    <div id="portal-advanced-search" class="hiddenStructure">
        <a href="../search_form.html" accesskey="5">
            Advanced Search&hellip;
        </a>
    </div>

</div>
<a id="portal-logo" title="Site" accesskey="1" href="../index.html">
    <img src="../logo.png" alt="Site" title="Site" height="100" width="500" /></a>

    <h5 class="hiddenStructure">Sections</h5>

    <ul id="portal-globalnav">
<li id="portaltab-index_html" class="plain"><div class="left"></div><a href="../index.html" title="">Home</a><div class="right" /></li>
<li id="portaltab-about" class="plain"><div class="left"></div><a href="../about.html" title="">About ICBP</a><div class="right" /></li>
<li id="portaltab-icbp-centers" class="plain"><div class="left"></div><a href="../icbp-centers.html" title="">ICBP Centers</a><div class="right" /></li>
<li id="portaltab-resources" class="selected"><div class="left"></div><a href="../resources.html" title="">Resources</a><div class="right" /></li>
<li id="portaltab-icbp-news" class="plain"><div class="left"></div><a href="../icbp-news.html" title="">ICBP News</a><div class="right" /></li>
<li id="portaltab-education-and-outreach" class="plain"><div class="left"></div><a href="../education-and-outreach.html" title="">Education and Outreach</a><div class="right" /></li>

</ul>

</div>

                </div>
        </div>

        
        


        
    <div id="portal-columns" class="row">

        <div id="portal-column-content" class="cell width-full position-0">

            <div id="viewlet-above-content"><div id="portal-breadcrumbs">

    <span id="breadcrumbs-you-are-here">You
are here:</span>
    <span id="breadcrumbs-home">
        <a href="../index.html">Home</a>
        <span class="breadcrumbSeparator">
            &rsaquo;
            
        </span>
    </span>
    <span id="breadcrumbs-1" dir="ltr">
        
            <a href="../resources.html">Resources</a>
            <span class="breadcrumbSeparator">
                &rsaquo;
                
            </span>
            
         
    </span>
    <span id="breadcrumbs-2" dir="ltr">
        
            
            
            <span id="breadcrumbs-current">Software</span>
         
    </span>

</div>
</div>

            
                <div class="">

                    

                    

    <dl class="portalMessage info" id="kssPortalMessage" style="display:none">
        <dt>Info</dt>
        <dd></dd>
    </dl>



                    
                    <table cellpadding="0" cellspacing="0" border="0" id="contentTable">
                    <tr>
                        <td valign="top">
                            <div id="content" style="">
                                
    
                                
    
                                 <div id="viewlet-above-content-title"></div>
                                 
                                     
                                     
            
                <h1 id="parent-fieldname-title" class="documentFirstHeading">
                    Software
                </h1>
            
            
        
                                 
                                 <div id="viewlet-below-content-title">
</div>
    
                                 
                                     
                                     
            
                
            
            
        
                                 
    
                                 <div id="viewlet-above-content-body">

</div>
                                 <div id="content-core">
                                     
    
        
        <div class="" id="parent-fieldname-text-4bab9f6f-8be8-4545-a72f-97c9b6614976">
            <p><strong>The Broad Institute/Dana Farber Cancer Institute<br /> Principal Investigator: Todd Golub, M.D.</strong></p>
<p><strong>GenePattern</strong><br /> PI: Jill P. Mesirov, Ph.D.<br /> Contact: <a href="mailto:gp-help@broadinstitute.org" title="gp-help@broadinstitute.org">gp-help@broadinstitute.org</a><br /> Web Link: <a href="http://www.genepattern.org/" title="http://www.genepattern.org">http://www.genepattern.org</a></p>
<p>GenePattern is a powerful genomic analysis platform that provides access to more than 180 tools for gene expression analysis, proteomics, SNP analysis, flow cytometry, RNA-seq analysis, and common data processing tasks. A web-based interface provides easy access to these tools and allows the creation of multi-step analysis pipelines that enable reproducible in silico research.</p>
<p><strong>Integrative Genomics Viewer (IGV)</strong><br /> PI: Jill P. Mesirov, Ph.D.<br /> Contact: <a href="mailto:igv-help@broadinstitute.org" title="igv-help@broadinstitute.org">igv-help@broadinstitute.org</a><br /> Web Link: <a href="http://www.broadinstitute.org/igv" target="_blank" title="http://www.broadinstitute.org/igv">http://www.broadinstitute.org/igv</a></p>
<p>IGV is a high-performance visualization tool for interactive exploration of large, integrated genomic datasets. It supports a wide variety of data types including alignments for standard and short-read sequence data, microarrays, genomic variation, mutation, methylation and histone modification, and genomic annotations. Users can navigate quickly and easily from whole-genome to single base pair views as well as panning across a region of genomic interest. IGV supports the simultaneous viewing of hundreds to thousands of samples, providing extensive sorting, filtering, and selection capabilities.</p>
<p><strong>Connectivity Map (CMAP)</strong><br /> PI: Todd Golub, M.D.<br /> Contact: <a href="mailto:cmap-help@broadinstitute.org" title="cmap-help@broadinstitute.org">cmap-help@broadinstitute.org</a><br /> Web Link: <a href="http://www.connectivitymap.org/cmap/" target="_blank" title="http://www.connectivitymap.org/cmap/">http://www.connectivitymap.org/cmap/</a></p>
<p>The Connectivity Map allows researchers to screen compounds against genome-wide disease signatures, rather than a pre-selected set of target genes. Drugs are paired with diseases using sophisticated pattern-matching methods with a high level of resolution and specificity.</p>
<p><strong>Gene Set Enrichment Analysis (GSEA)</strong><br /> PI: Jill P. Mesirov, Ph.D.<br /> Contact: <a href="mailto:gsea@broadinstitute.org" title="gsea@broadinstitute.org">gsea@broadinstitute.org</a><br /> Web Link: <a href="http://www.broadinstitute.org/gsea/" target="_blank" title="http://www.broadinstitute.org/gsea/">http://www.broadinstitute.org/gsea/</a></p>
<p>A computational method that determines whether an a priori defined set of genes shows statistically significant, concordant differences between two biological states (e.g., phenotypes).</p>
<p><strong>GeneCruiser</strong><br /> PI: Jill P. Mesirov, Ph.D.<br /> Contact: <a href="mailto:genecruiser@broad.mit.edu" title="genecruiser@broad.mit.edu">genecruiser@broad.mit.edu</a><br /> Web Link: <a href="http://genecruiser.broadinstitute.org/genecruiser3/" target="_blank" title="http://genecruiser.broadinstitute.org/genecruiser3/">http://genecruiser.broadinstitute.org/genecruiser3/</a></p>
<p>GeneCruiser provides integrated access to genomic information available from public data sources. Users can convert between Affymetrix gene expression and SNP chip microarrays and corresponding annotations in UniGene, Entrez Gene, and other genomic resources.</p>
<p><strong>GenomeSpace</strong><br /> PI: Jill Mesirov, Ph.D.<br /> Contact: <a href="mailto:gs-help@broadinstitute.org" title="gs-help@broadinstitute.org">gs-help@broadinstitute.org</a><br /> Web Link: <a href="http://www.genomespace.org/" target="_blank" title="http://www.genomespace.org">http://www.genomespace.org</a></p>
<p>GenomeSpace brings together diverse computational tools, enabling scientists without programming skills to easily combine their capabilities. It aims to offer a common space to create, manipulate and share an ever-growing range of genomic analysis tools.</p>
<p>Initially seeded with several popular genomics tools, GenomeSpace will provide a common workspace that enhances the tools’ interoperability. As before, all tools available within GenomeSpace will remain accessible as individual packages from their respective development groups for use as stand-alone software.</p>
<p><strong>Tumorscape</strong><br /> PI: Matthew Meyerson, Ph.D.<br /> Contact: Michael Reich (<a href="mailto:michaelr@broadinstitute.org" title="michaelr@broadinstitute.org">michaelr@broadinstitute.org</a>)<br /> Web Link: <a href="http://www.tumorscape.org/" target="_blank" title="http://www.tumorscape.org">http://www.tumorscape.org</a></p>
<p>Tumorscape facilitates the use and understanding of high resolution copy number data amassed from multiple cancer types.</p>
<p><strong>Broad-Novartis Cell Line Encyclopedia</strong><br /> PI: Levi Garraway, Ph.D.<br /> Contact: Michael Reich (<a href="michaelr%40broadinstitute.html" title="michaelr@broadinstitute.org">michaelr@broadinstitute.org</a>)<br /> Web Link: <a href="http://www.broadinstitute.org/ccle" target="_blank" title="http://www.broadinstitute.org/ccle">http://www.broadinstitute.org/ccle</a></p>
<p>The Cancer Cell Line Encyclopedia (CCLE) project is a collaboration between the Broad Institute, the Novartis Institutes for Biomedical Research and the Genomics Institute of the Novartis Research Foundation to conduct a detailed genetic and pharmacologic characterization of a large panel of human cancer models, to develop integrated computational analyses that link distinct pharmacologic vulnerabilities to genomic patterns and to translate cell line integrative genomics into cancer patient stratification. The CCLE provides public access analysis and visualization of DNA copy number, mRNA expression and mutation data for about 1000 cell lines.</p>
<p><strong>GENE-E</strong><br /> PI: Todd Golub, M.D.<br /> Contact: Michael Reich (<a href="mailto:michaelr@broadinstitute.org" title="michaelr@broadinstitute.org">michaelr@broadinstitute.org</a>)<br /> Web Link: <a href="http://www.broadinstitute.org/cancer/software/GENE-E/" target="_blank" title="http://www.broadinstitute.org/cancer/software/GENE-E/">http://www.broadinstitute.org/cancer/software/GENE-E/</a></p>
<p>Gene-E allows the rapid visual exploration of datasets derived from RNAi and chemical screens.</p>
<hr />
<p></p>
<p><strong>Columbia University<br /> Principal Investigator: Andrea Califano, Ph.D.</strong></p>
<p><strong>ARACNe</strong><br /> Contact: Mukesh Bansal, Ph.D., Columbia University<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/califanolab/index.php?title=Aracne_registration&amp;redirect=no" target="_blank" title="http://wiki.c2b2.columbia.edu/califanolab/index.php?title=Aracne_registration&redirect=no">http://wiki.c2b2.columbia.edu/califanolab/index.php?title=Aracne_registration&amp;redirect=no</a></p>
<p>ARACNE (Algorithm for the Reconstruction of Accurate Cellular Networks) is an algorithm for inferring transcriptional gene regulatory networks from a set of microarray experiments. The method uses mutual information to identify genes that are co-expressed and then applies an information theoretic approach, the data processing inequality, to eliminate the vast majority of indirect interactions typically inferred by pairwise analysis. ARACNE is specifically designed to scale up to the complexity of regulatory networks in mammalian cells, yet general enough to address a wider range of network deconvolution problems.</p>
<p><strong>MINDY2</strong><br /> Contact: Mukesh Bansal, Ph.D., Columbia University<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/califanolab/index.php/Software/MINDY2" target="_blank" title="http://wiki.c2b2.columbia.edu/califanolab/index.php/Software/MINDY2">http://wiki.c2b2.columbia.edu/califanolab/index.php/Software/MINDY2</a></p>
<p>Given a transcription factor of interest, MINDY uses a large set of gene expression profile data to identify potential post-transcriptional modulators of the transcription factor's activity. MINDY is based on a three-way statistical interaction model that captures the post-transcriptional regulatory event where the ability of a transcription factor to activate/repress its target genes is monotonically controlled by a potential modulator gene.</p>
<p><strong>SPLASH</strong><br /> Contact: Andrea Califano, Ph.D., Columbia University<br /> Web Link: <a href="http://amdec-bioinfo.cu-genome.org/html/Splash.html" target="_blank" title="http://amdec-bioinfo.cu-genome.org/html/Splash.html">http://amdec-bioinfo.cu-genome.org/html/Splash.html</a></p>
<p>SPLASH (Structural Pattern Localization Analysis by Sequential Histograms) is a deterministic pattern discovery algorithm which can find sparse amino or nucleic acid patterns matching identically or similarly in a set of protein or DNA sequences. Sparse patterns of any length, up to the size of the input sequence can be discovered without significant loss in performance. Splash is extremely efficient and embarrassingly parallel by nature.</p>
<p><strong>CLEANER</strong><br /> Contact: Mariano Alvarez<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/califanolab/index.php/Cleaner" target="_blank" title="http://wiki.c2b2.columbia.edu/califanolab/index.php/Cleaner">http://wiki.c2b2.columbia.edu/califanolab/index.php/Cleaner</a></p>
<p>R-system software package for the assembly of informative, transcript-specific probe-clusters for Affymetrix expression microarrays.</p>
<p><strong>geWorkbench</strong><br /> Contact: Kenneth Smith, Ph.D., Columbia University<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/workbench/index.php/Home" target="_blank" title="http://wiki.c2b2.columbia.edu/workbench/index.php/Home">http://wiki.c2b2.columbia.edu/workbench/index.php/Home</a></p>
<p>geWorkbench is a bioinformatics platform that provides users with an integrated suite of genomics tools. It is built on an open-source, extensible architecture that promotes interoperability and simplifies the development of new as well as the incorporation of pre-existing components. The resulting system provides seamless access to a multitude of both local and remote data and computational services through an integrated environment that offers a unified user experience. Over 70 data analysis and visualization components have been developed for the framework, covering a wide range of genomics domains including gene expression, sequence, structure, and network data.</p>
<p><strong>REDUCE Suite v2.0</strong><br /> Contact: Harmen Bussemaker, Ph.D., Columbia University<br /> Web Link: <a href="http://bussemaker.bio.columbia.edu/software/REDUCE/" target="_blank" title="http://bussemaker.bio.columbia.edu/software/REDUCE/">http://bussemaker.bio.columbia.edu/software/REDUCE/</a></p>
<p>The REDUCE Suite v2.0 is a set of software tools to model the regulation of gene expression by transcription factors (TF). By directly correlating genome-wide mRNA expression or TF binding data (e.g. ChIP-chip) with associated nucleotide sequences, the REDUCE Suite can discover the sequence-specific binding affinity of a TF from a single experiment, using all measurements simultaneously, and without using any "background" sequence model. The REDUCE suite has its origin in the REDUCE algorithm of Bussemaker, Li, and Siggia, Nature Genetics (2001), which pioneered the use of motif-based linear regression model to discover cis-regulatory elements (motifs) and infer condition-specific transcription factor activities from a single genomewide mRNA expression profile.</p>
<p><strong>TranscriptionDetector </strong><br /> Contact: Harmen Bussemaker, Ph.D., Columbia University<br /> Web Link: <a href="http://bussemaker.bio.columbia.edu/software/TranscriptionDetector/" target="_blank" title="http://bussemaker.bio.columbia.edu/software/TranscriptionDetector/">http://bussemaker.bio.columbia.edu/software/TranscriptionDetector/</a></p>
<p>TranscriptionDetector is a tool for finding probes measuring significantly expressed loci in a genomic array experiment. This software was developed in Harmen Bussemaker's Lab at Columbia University by Gabor Halasz and Marinus van Batenburg, with the source code ported from Perl to C by Xiang-Jun Lu. TranscriptionDetector was originally created to analyze the transcriptome of Drosophila melanogaster. The TranscriptionDetector algorithm requires that a set of negative control probes (NCPs) be included on each microarray used in the study. The significance of a data probe is evaluated relative to these NCPs, which represent non-specific binding.</p>
<p><strong>T-profiler</strong><br /> Contact: Harmen Bussemaker, Ph.D., Columbia University<br /> Web Link: <a href="http://www.t-profiler.org/" target="_blank" title="http://www.t-profiler.org/">http://www.t-profiler.org/</a></p>
<p>One of the key challenges in the analysis of gene expression data is how to relate the expression level of individual genes to the underlying transcriptional programs and cellular state. The T-profiler tool hosted on this website uses the t-test to score changes in the average activity of pre-defined groups of genes. The gene groups are defined based on Gene Ontology categorization, ChIP-chip experiments, upstream matches to a consensus transcription factor binding motif, and location on the same chromosome, respectively. If desired, an iterative procedure can be used to select a single, optimal representative from sets of overlapping gene groups. A jack-knife procedure is used to make calculations more robust against outliers. T-profiler makes it possible to interpret microarray data in a way that is both intuitive and statistically rigorous, without the need to combine experiments or choose parameters. Currently, gene expression data from Saccharomyces <em>cerevisiae</em> and <em>Candida albicans</em> are supported.</p>
<p><strong>cryoEM Refinement Tools</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:cryoEM" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:cryoEM">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:cryoEM</a></p>
<p>Programs and protocols that can be used to refine atomic structures, either experimentally determined or computationally constructed, within cryo-electron microscopy (cryoEM) density maps of macromolecular complexes.</p>
<p><strong>DelPhi</strong><br /> Contact : Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:DelPhi" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:DelPhi">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:DelPhi</a></p>
<p>DelPhi provides numerical solutions to the Poisson-Boltzmann equation (both linear and nonlinear form) for molecules of arbitrary shape and charge distribution. The current version is fast (the best relaxation parameter is estimated at run time), accurate (calculation of the electrostatic free energy is less dependent on the resolution of the lattice) and can handle extremely high lattice dimensions. It also includes flexible features for assigning different dielectric constants to different regions of space and treating systems containing mixed salt solutions.</p>
<p><strong>GRASP</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="honig_software%40c2b2.columbia.html" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:GRASP" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:GRASP">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:GRASP</a></p>
<p>GRASP is a molecular visualization and analysis program. It is particularly useful for the display and manipulation of the surfaces of molecules and their electrostatic properties. For <strong>SGI machines ONLY</strong>.</p>
<p><strong>GRASP2</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:GRASP2" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:GRASP2">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:GRASP2</a></p>
<p>GRASP2 is an updated version of the GRASP program used for macromolecular structure and surface visualization. It is written for the Windows operating system and contains a large number of new features and scientific tools.</p>
<p><strong>Jackal 1.5</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="honig_software%40c2b2.columbia.html" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edum</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Jackal" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Jackal">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Jackal</a></p>
<p>Jackal is a collection of programs designed for the modeling and analysis of protein structures. Its core program is a versatile homology modeling package nest. Some of its many components include</p>
<ul>
<li><a href="http://bhapp.c2b2.columbia.edu/software/Jackal/Jackalmanual.htm#nest" target="_blank" title="nest">nest</a>for building homology models based on 1) a single sequence-template alignment, 2) from multiple templates for the entire structure, or 3) from different templates used for different regions of the structure. It also carries out energy based structure refinement and can change an alignment based on energetic considerations.</li>
<li><a href="http://bhapp.c2b2.columbia.edu/software/Jackal/Jackalmanual.htm#loopy" target="_blank" title="loopy">loopy</a>for loop prediction including mutation, insertion and deletion in a loop region.</li>
<li><a href="http://bhapp.c2b2.columbia.edu/software/Jackal/Jackalmanual.htm#scap" target="_blank" title="scap">scap</a> for prediction of side-chain conformations on a given backbone. It can be used to predict the conformations of one or more residues that have been “mutated” in a particular structure.</li>
</ul>
<p><strong>PrISM</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PrISM" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PrISM">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PrISM</a></p>
<p>An integrated computational system with tools for protein modeling and sequence and structure analysis.</p>
<p><strong>Protein-DNA Interface Alignment Software:</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Protein-DNA_Interface_Alignment" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Protein-DNA_Interface_Alignment">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Protein-DNA_Interface_Alignment</a></p>
<p>An algorithm to perform interface alignment between any two protein-DNA complexes based on their corresponding PDB files.</p>
<p><strong>Protein-DNA Modeling Interface Software: </strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="honig_software%40c2b2.columbia.html" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Protein-DNA_Modeling_Interface" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Protein-DNA_Modeling_Interface">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Protein-DNA_Modeling_Interface</a></p>
<p>The program can be used predict the conformation of the sidechains within a protein-DNA binding interface.</p>
<p><strong>SURFace Algorithms</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="honig_software%40c2b2.columbia.html" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:SURFace_Algorithms" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:SURFace_Algorithms">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:SURFace_Algorithms</a></p>
<p>Programs that calculate solvent accessible surface area and curvature corrected solvent accessible surface area. The accessible surface output is calculated for the whole molecule, per residue, and per atom.</p>
<p><strong>VISTAL</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Vistal" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Vistal">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Vistal</a></p>
<p>A two-dimensional visualization tool for structural alignments. VISTAL describes structures as a series of secondary structure elements, and places matched residues one on top of each other colored according to the three-dimensional distance of their Ca atoms.</p>
<p><strong>PredUs</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PredUs" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PredUs">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PredUs</a></p>
<p>PredUs (Prediction of Protein Interfaces Using Structural Alignment) is a tool to predict protein-protein interface based on interface conservation among structurally similar protein neighbors.</p>
<p><strong>PUDGE</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>) <br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PUDGE" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PUDGE">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:PUDGE</a></p>
<p>This protein structure prediction server provides a convenient means of integrating tools that may be used at different stages of the structural prediction process.</p>
<p><strong>Mark-Us</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Mark-Us" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Mark-Us">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Mark-Us</a></p>
<p>Mark-Us is a web server to assist the assessment of the biochemical function for a given protein structure. Mark-Us identifies related protein structures and sequences, detects protein cavities, and calculates the surface electrostatic potentials and amino acid conservation profile. The results can be browsed by an interactive web interface that allows the integration of Gene Ontology terms, UniProt features, and the Enzyme Classification.</p>
<p><strong>Target Explorer</strong><br /> Contact Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Target_Explorer" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Target_Explorer">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Target_Explorer</a></p>
<p>A web based tool for automatic identification of new target genes for transcription factors in the Drosophila genome. Target Explorer automates the entire process from the creation of a customized library of binding sites for known transcription factors through the prediction and annotation of putative target genes that are potentially regulated by these factors.</p>
<p><strong>EDGI</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" target="_blank" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>) <br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:EDGI" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:EDGI">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:EDGI</a></p>
<p>EDGI (Enhancer Detection using only Genomic Information) integrates a traditional motif-discovery algorithm SPLASH with a novel local permutation-clustering algorithm PromoClust in order to identify large regulatory elements (e.g. enhancers) as evolutionarily conserved, order-independent clusters of short conserved motifs. EDGI relies on the evolutionary conservation of the binding sites for regulating transcription factors, without requiring knowledge of these transcription factors or their binding sites.</p>
<p><strong>Model Quality Assessment: Global and Local Quality Assessment of Protein Models</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Model_Quality_Assessment" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Model_Quality_Assessment">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Model_Quality_Assessment</a></p>
<p>Three protocols have been implemented for model quality assessment (MQA), including direct assessment of local qualities using statistical potentials and two machine-learning based protocols. The statistical potentials used here include newly developed solvation potentials, environmental potentials, pairwise potentials, hydrogen bonding potentials and backbone dihedral potentials. The normalized version of these potentials can be directly used to evaluate local qualities by plotting against residue numbers.</p>
<p><strong>STING Millennium</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" target="_blank" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://luna.bioc.columbia.edu/SMS/STINGm/" target="_blank" title="http://luna.bioc.columbia.edu/SMS/STINGm/">http://luna.bioc.columbia.edu/SMS/STINGm/</a></p>
<p>A web based suite of programs for comprehensive and simultaneous analysis of structure and sequence: nature and volume of atomic contacts of both intra- and inter- chain types, analysis of the quality of structure, residue conservation and its relationship with intra-chain contacts, functional parameters are also deciphered.</p>
<p><strong>SCREEN</strong><br /> Contact: Barry Honig, Ph.D., Columbia University (<a href="mailto:honig_software@c2b2.columbia.edu" title="honig_software@c2b2.columbia.edu">honig_software@c2b2.columbia.edu</a>)<br /> Web Link: <a href="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:SCREEN" target="_blank" title="http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:SCREEN">http://wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:SCREEN</a></p>
<p>A web server tool for identifying protein cavities and computing cavity attributes that can be applied for classification and analysis.</p>
<p><strong>Biolearn 1.0</strong><br /> Contact: Dana Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.c2b2.columbia.edu/danapeerlab/html/biolearn.html" target="_blank" title="http://www.c2b2.columbia.edu/danapeerlab/html/biolearn.html">http://www.c2b2.columbia.edu/danapeerlab/html/biolearn.html</a></p>
<p>Biolearn is a general package for applying probabilistic graphical models to biological applications. Biolearn release 1.0 is concentrated on structure learning for bayesian networks; future releases are expected to include other types of graphical models, support inference applications, and allow plug-and-play addition of new types of probability distributions, scoring functions and search algorithms.</p>
<p><strong>Genatomy</strong><br /> Contact: Dana Pe’er, Ph.D., Columbia University (<a href="mailto:genatomy@gmail.com" title="genatomy@gmail.com">genatomy@gmail.com</a>)<br /> Web Link: <a href="http://www.c2b2.columbia.edu/danapeerlab/html/genatomy.html" target="_blank" title="http://www.c2b2.columbia.edu/danapeerlab/html/genatomy.html">http://www.c2b2.columbia.edu/danapeerlab/html/genatomy.html</a></p>
<p>Genatomy is a visualizing tool for biological data, such as gene expression, genotypes, growth curves, copy number variation and more. It can be used to analyze the data mathematically and to study the biological aspects of the data and the results.</p>
<p><strong>Camelot</strong><br /> Contact: Dana Pe’er, Ph.D., Columbia University (<a href="mailto:camelot.program@gmail.com" title="camelot.program@gmail.com">camelot.program@gmail.com</a>)<br /> Web Link: <a href="http://www.c2b2.columbia.edu/danapeerlab/html/camelot.html" target="_blank" title="http://www.c2b2.columbia.edu/danapeerlab/html/camelot.html">http://www.c2b2.columbia.edu/danapeerlab/html/camelot.html</a></p>
<p>CAusal Modeling with Expression Linkage for cOmplex Traits (Camelot) provides a framework to both model complex traits and identify the potential underlying causal factors. Briefly, Camelot takes genotype, gene expression and phenotype data as input, and output a linear regression model that uses genotype and expression to predict phenotype. Powered by regularized linear regression, Camelot aims to choose features (either genotype or gene expression) that are predictive (and likely causal) to the phenotype. In addition, Camelot also provides functions to prioritize genes residing within a locus that is associated with the trait.</p>
<p><strong>JISTIC</strong><br /> Contact: Dana Pe’er, Ph.D., Columbia University <br /> Web Link: <a href="http://www.c2b2.columbia.edu/danapeerlab/html/jistic.html" target="_blank" title="http://www.c2b2.columbia.edu/danapeerlab/html/jistic.html">http://www.c2b2.columbia.edu/danapeerlab/html/jistic.html</a></p>
<p>JISTIC is a tool for analyzing datasets of genome-wide copy number variation to identify driver aberrations in cancer. JISTIC is an improvement over the widely used GISTIC algorithm.</p>
<p><strong>LitVAn</strong><br /> Contact: Dana Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://litvan.bio.columbia.edu/" target="_blank" title="http://litvan.bio.columbia.edu/">http://litvan.bio.columbia.edu/</a></p>
<p>Literature Vector Analysis, is an algorithm that searches forover-represented terms in papers associated with genes in a gene set. LitVAn uses a manually curated database (NCBI Gene) to connect genes with terms from the complete text of more than 70,000 published scientific articles.</p>
<p><strong>CONEXIC</strong><br /> Contact: Dana Pe’er, Ph.D., Columbia University (<a href="mailto:conexic@gmail.com" title="conexic@gmail.com">conexic@gmail.com</a>)<br /> Web Link: <a href="http://www.c2b2.columbia.edu/danapeerlab/html/conexic.html" target="_blank" title="http://www.c2b2.columbia.edu/danapeerlab/html/conexic.html">http://www.c2b2.columbia.edu/danapeerlab/html/conexic.html</a></p>
<p>COpy Number and EXpression In Cancer (CONEXIC) is an algorithm that integrates matched copy number (amplifications and deletions) and gene expression data from tumor samples to identify driving mutations and the processes they influence. CONEXIC is inspired by Module Networks (Segal et al, 2003), but has been augmented by a number of critical modifications that make it suitable for identifying drivers. CONEXIC uses a score-guided search to identify the combination of modulators that best explains the behavior of a gene expression module across tumor samples and searches for those with the highest score within the amplified or deleted region.</p>
<p><strong>SIXPAC</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~snehitp/sixpac/" target="_blank" title="http://www.cs.columbia.edu/~snehitp/sixpac/">http://www.cs.columbia.edu/~snehitp/sixpac/</a></p>
<p>SIXPAC (Search for Interactions is Probably Approximately Complete) is an efficient, scalable search algorithm that finds synergy between pairs of physically unlinked SNPs (genome-wide) in large case-control datasets.</p>
<p><strong>Germline</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University <br /> Web Link: <a href="http://www1.cs.columbia.edu/~gusev/germline/" target="_blank" title="http://www1.cs.columbia.edu/~gusev/germline/">http://www1.cs.columbia.edu/~gusev/germline/</a></p>
<p>GERMLINE is an algorithm for discovering long shared segments of Identity by Descent (IBD) between pairs of (unrelated) individuals in a large population.</p>
<p><strong>DASH</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www1.cs.columbia.edu/~gusev/dash/" target="_blank" title="http://www1.cs.columbia.edu/~gusev/dash/">http://www1.cs.columbia.edu/~gusev/dash/</a></p>
<p>DASH (DASH Associates Shared Haplotypes) is a tool for detecting association to clusters of IBD segments detected by Germline. It builds upon pairwise IBD shared segments to infer clusters of IBD individuals.</p>
<p><strong>HLAIBD</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/hla_ibd/index.html" target="_blank" title="http://www.cs.columbia.edu/~itsik/hla_ibd/index.html">http://www.cs.columbia.edu/~itsik/hla_ibd/index.html</a></p>
<p>The Human Leukocyte Antigen (HLA) genes play a major role in adaptive immune response and are used to differentiate self cells from non-self cells. HLA genes are hyper variable with nearly every locus containing over a dozen alleles. This variation plays an important role in autoimmune disease and organ transplantation. HLA typing by serological methods is time consuming and expensive. HLABID is a new computational method to infer per-locus HLA types using shared segments Identical By Descent (IBD), inferred from genotype data.</p>
<p><strong>INFOSTIP</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/INFOSTIP/readme.html" target="_blank" title="http://www.cs.columbia.edu/~itsik/INFOSTIP/readme.html">http://www.cs.columbia.edu/~itsik/INFOSTIP/readme.html</a></p>
<p>A tool for selecting INdividual FOr Sequencing by Total Information Potential, based on GERMLINE output.</p>
<p><strong>ShareViz</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/sharevizWeb/shareviz.html" target="_blank" title="http://www.cs.columbia.edu/~itsik/sharevizWeb/shareviz.html">http://www.cs.columbia.edu/~itsik/sharevizWeb/shareviz.html</a></p>
<p>Shareviz is a tool for graphical visualization of hidden relatedness based on GERMLINE output.</p>
<p><strong>HADiT</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/HADiT/HADiT.html" target="_blank" title="http://www.cs.columbia.edu/~itsik/HADiT/HADiT.html">http://www.cs.columbia.edu/~itsik/HADiT/HADiT.html</a></p>
<p>HADiT (Haplotype Amplification Distortion in Tumors) is a tool for computing and visualizing allelic distortion in tumor SNP data. Implements the Amplification Distortion Test (ADT).</p>
<p><strong>HATS</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/HATS/HATS.html" target="_blank" title="http://www.cs.columbia.edu/~itsik/HATS/HATS.html">http://www.cs.columbia.edu/~itsik/HATS/HATS.html</a></p>
<p>HATS is a tool for calling the amplified alleles and constructing the amplified haplotype within called tumor amplicons.</p>
<p><strong>PoolDesign</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/Pooling/pooldesign.jar" target="_blank" title="http://www.cs.columbia.edu/~itsik/Pooling/pooldesign.jar">http://www.cs.columbia.edu/~itsik/Pooling/pooldesign.jar</a></p>
<p>PoolDesign is a tool for designing overlapping pools for variant-carrier identification.</p>
<p><strong>Opera</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/OPERA/" target="_blank" title="http://www.cs.columbia.edu/~itsik/OPERA/">http://www.cs.columbia.edu/~itsik/OPERA/</a></p>
<p>OPERA is a tool for power estimation and design of whole-genome resequencing projects aimed at rare variant associations.</p>
<p><strong>Zinfandel</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/zinfandel/" target="_blank" title="http://www.cs.columbia.edu/~itsik/zinfandel/">http://www.cs.columbia.edu/~itsik/zinfandel/</a></p>
<p>Zinfandel is a tool for detecting CNVs from low-pass WGS data.</p>
<p><strong>Inference of Modules associated to eQTLs</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/IMR_eQTL2/Inference_of_modules_associated_to_eQTLs.htm" target="_blank" title="http://www.cs.columbia.edu/~itsik/IMR_eQTL2/Inference_of_modules_associated_to_eQTLs.htm">http://www.cs.columbia.edu/~itsik/IMR_eQTL2/Inference_of_modules_associated_to_eQTLs.htm</a></p>
<p>A tool for constructing modules regulated by eQTLs using gene expression and SNP variation data across multiple individuals.</p>
<p><strong>CONDEX (condr)</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://code.google.com/p/condr/" target="_blank" title="http://code.google.com/p/condr/">http://code.google.com/p/condr/</a></p>
<p>HMM Model to detect copy number variants in exome sequence data</p>
<p><strong>MutaGeneSys</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www1.cs.columbia.edu/~jds1/MutaGeneSys/" target="_blank" title="http://www1.cs.columbia.edu/~jds1/MutaGeneSys/">http://www1.cs.columbia.edu/~jds1/MutaGeneSys/</a></p>
<p>MutaGeneSys uses genome-wide genotype data to estimate individual disease susceptibility. It integrates OMIM and HapMap-based single- and multi-marker predictors of causal SNPs to report the collection of associated alleles in a whole-genome-typed individual.</p>
<p><strong>Xplorigin</strong><br /> Contact: Itsik Pe’er, Ph.D., Columbia University<br /> Web Link: <a href="http://www.cs.columbia.edu/~itsik/Xplorigin/Xplorigin.htm" target="_blank" title="http://www.cs.columbia.edu/~itsik/Xplorigin/Xplorigin.htm">http://www.cs.columbia.edu/~itsik/Xplorigin/Xplorigin.htm</a></p>
<p>A tool for parsing population ancestry of admixed individuals.</p>
<p><strong>FASD</strong><br /> Contact: Raul Rabadan, Ph.D., Columbia University<br /> Web Link: <a href="http://projects.dbmi.columbia.edu/fasd/index.php" target="_blank" title="http://projects.dbmi.columbia.edu/fasd/index.php">http://projects.dbmi.columbia.edu/fasd/index.php</a></p>
<p>Frequency Analysis of Sequence Data (<em>FASD</em>)</p>
<p><strong>TEISER </strong><br /> Contact: Saeed Tavazoie, Ph.D., Columbia University (<a href="mailto:support@tavazoielab.org" title="support@tavazoielab.org">support@tavazoielab.org</a>)<br /> Web Link: <a href="https://tavazoielab.c2b2.columbia.edu/TEISER/" target="_blank" title="https://tavazoielab.c2b2.columbia.edu/TEISER/">https://tavazoielab.c2b2.columbia.edu/TEISER/</a></p>
<p>TEISER is a de-novo motif discovery tool for finding informative structural elements in RNA.</p>
<p><strong>FIRE </strong><br /> Contact: Saeed Tavazoie, Ph.D., Columbia University (<a href="mailto:support@tavazoielab.org" title="support@tavazoielab.org">support@tavazoielab.org</a>)<br /> Web Link: <a href="https://tavazoielab.c2b2.columbia.edu/FIRE/" target="_blank" title="https://tavazoielab.c2b2.columbia.edu/FIRE/"></a></p>
<p>FIRE is a motif discovery and characterization program based on mutual information.</p>
<p><strong>EVE </strong><br /> Contact: Saeed Tavazoie, Ph.D., Columbia University (<a href="mailto:support@tavazoielab.org" title="support@tavazoielab.org">support@tavazoielab.org</a>)<br /> Web Link: <a href="https://tavazoielab.c2b2.columbia.edu/EVE/" target="_blank" title="https://tavazoielab.c2b2.columbia.edu/EVE/">https://tavazoielab.c2b2.columbia.edu/EVE/</a></p>
<p>EVE is a simulation framework for microbial populations in complex dynamic environments</p>
<p><strong>iPAGE </strong><br /> Contact: Saeed Tavazoie, Ph.D., Columbia University (<a href="mailto:support@tavazoielab.org" title="support@tavazoielab.org">support@tavazoielab.org</a>)<br /> Web Link: <a href="https://tavazoielab.c2b2.columbia.edu/iPAGE/" target="_blank" title="https://tavazoielab.c2b2.columbia.edu/iPAGE/">https://tavazoielab.c2b2.columbia.edu/iPAGE/</a></p>
<p>iPAGE a functional and categorical enrichment program based on mutual information</p>
<p><strong>FIRE-pro </strong><br /> Contact: Saeed Tavazoie, Ph.D., Columbia University (<a href="mailto:support@tavazoielab.org" title="support@tavazoielab.org">support@tavazoielab.org</a>)<br /> Web Link: <a href="https://tavazoielab.c2b2.columbia.edu/FIRE-pro/" target="_blank" title="https://tavazoielab.c2b2.columbia.edu/FIRE-pro/">https://tavazoielab.c2b2.columbia.edu/FIRE-pro/</a></p>
<p>FIRE-pro is a motif discovery and characterization program for proteins based on mutual information.</p>
<p><strong>ADOMETA</strong><br /> Contact: Dennis Vitkup, Ph.D., Columbia University<br /> Web Link: <a href="http://vitkuplab.c2b2.columbia.edu/adometa/adometa.html" target="_blank" title="http://vitkuplab.c2b2.columbia.edu/adometa/adometa.html">http://vitkuplab.c2b2.columbia.edu/adometa/adometa.html</a></p>
<p>ADOMETA is a bioinformatics resource designed to predict genes for <a href="http://vitkuplab.c2b2.columbia.edu/adometa/introduction.html#orphan_activity" target="_blank" title="orphan metabolic activities">orphan metabolic activities</a> - known biochemical activities not currently assigned to genes in some or all organisms. ADOMETA returns a ranked list of genes likely to catalyze a given metabolic activity in a selected organism.</p>
<hr />
<p></p>
<p><strong>Georgetown University<br /> Principal Investigator: Robert Clarke, Ph.D.</strong></p>
<p><strong>JigCell</strong><br /> Contact: Subha Madhavan (<a href="mailto:sm696@georgetown.edu" title="sm696@georgetown.edu">sm696@georgetown.edu</a>)<br /> Web Link: <a href="http://jigcell.cs.vt.edu/" target="_blank" title="http://jigcell.cs.vt.edu/">http://jigcell.cs.vt.edu/</a></p>
<p>Biological reaction-based modeling and simulation tool</p>
<p><strong>SAIC</strong><br /> Contact: Subha Madhavan (<a href="mailto:sm696@georgetown.edu" title="sm696@georgetown.edu">sm696@georgetown.edu</a>)<br /> Web Link: <a href="http://www.cbil.ece.vt.edu/software.htm" target="_blank" title="http://www.cbil.ece.vt.edu/software.htm">http://www.cbil.ece.vt.edu/software.htm</a></p>
<p>SAIC is an analytic tool for characterizing and assessing the statistical significance of recurrent CNA units, providing useful information to study the landscape of cancer genomes. The concepts of CNA unit and iterative permutation are relatively simple to interpret, yet still convey considerable novel mathematical insights into data structure and bias correction.</p>
<p><strong>DDN</strong><br /> Contact: Subha Madhavan (<a href="mailto:sm696@georgetown.edu" title="sm696@georgetown.edu">sm696@georgetown.edu</a>)<br /> Web Link: <a href="http://www.cbil.ece.vt.edu/software.htm" target="_blank" title="http://www.cbil.ece.vt.edu/software.htm">http://www.cbil.ece.vt.edu/software.htm</a></p>
<p>DDN is an analytical tool for detecting and visualizing statistically significant topological changes in dependence networks between two biological conditions. DDN enables differential network analysis and provides an alternative way for defining network biomarkers predictive of phenotypes. DDN also serves as a useful systems biology tool for users across biomedical research communities to infer how genetic, epigenetic or environment variables may affect biological networks and clinical phenotypes.</p>
<p><strong>PUGSVM</strong><br /> Contact: Subha Madhavan (<a href="mailto:sm696@georgetown.edu" title="sm696@georgetown.edu">sm696@georgetown.edu</a>)<br /> Web Link: <a href="http://www.cbil.ece.vt.edu/software.htm" target="_BLANK" title="http://www.cbil.ece.vt.edu/software.htm">http://www.cbil.ece.vt.edu/software.htm</a></p>
<p>PUGSVM is an analytical tool for multiclass gene selection and classification. PUGSVM addresses the problem of imbalanced class separability, small sample size and high gene space dimensionality, where multiclass gene markers are defined by the union of one-versus-everyone phenotypic up-regulated genes, and used by a well-matched one-versus-rest support vector machine (OVRSVM). PUGSVM provides a simple yet more accurate strategy to identify statistically-reproducible mechanistic marker genes for characterization of heterogeneous cancers.</p>
<p><strong>CAM-CM</strong><br /> Contact: Subha Madhavan (<a href="mailto:sm696@georgetown.edu" title="sm696@georgetown.edu">sm696@georgetown.edu</a>)<br /> Web Link: <a href="http://www.cbil.ece.vt.edu/software.htm" target="_blank" title="http://www.cbil.ece.vt.edu/software.htm">http://www.cbil.ece.vt.edu/software.htm</a></p>
<p>CAM-CM is analytic tool for deconvolving dynamic imaging series in heterogeneous tumor that can improve quantitations of vascular pharmacokinetics in many biological contexts. Since intratumor heterogeneity is often manifested by vascular compartments of distinct pharmacokinetics that cannot be resolved directly by in vivo dynamic imaging, revealed characteristic intratumor vascular heterogeneity and therapeutic responses that were otherwise undetectable.</p>
<p><strong>mSD</strong><br /> Contact: Subha Madhavan (<a href="mailto:sm696@georgetown.edu" title="sm696@georgetown.edu">sm696@georgetown.edu</a>)<br /> Web Link: <a href="http://www.cbil.ece.vt.edu/software.htm" target="_blank" title="http://www.cbil.ece.vt.edu/software.htm">http://www.cbil.ece.vt.edu/software.htm</a></p>
<p>Motif-guided sparse decomposition (mSD) is an integrated algorithm for regulatory network identification, which combines gene expression data and DNA motif information to find gene clusters of co-regulation and co-regulation. The mSD algorithm also features a sparse decomposition method for regulation strength estimation so as to define transcript factors’ target genes.</p>
<p><strong>cancer Bench-to-Bedside (caB2B)</strong><br /> Contact: Baris Suzek (<a href="mailto:bes23@georgetown.edu" title="bes23@georgetown.edu">bes23@georgetown.edu</a>)<br /> Web Link: <a href="https://wiki.uis.georgetown.edu/display/caB2B" target="_blank" title="https://wiki.uis.georgetown.edu/display/caB2B">https://wiki.uis.georgetown.edu/display/caB2B</a></p>
<p>cancer Bench-to-Beside (caB2B) is an open-source, secure query tool that allows translational research scientists to search and combine data from virtually any data service available on caBIG.</p>
<hr />
<p></p>
<p><strong>Massachusetts Institute of Technology (MIT)<br /> Principal Investigator: Doug Lauffenburger, Ph.D.</strong></p>
<p><strong>NetPhorest</strong><br /> Contact: <a href="mailto:rune.linding@gmail.com" title="rune.linding@gmail.com">rune.linding@gmail.com</a><br /> Web Link: <a href="http://netphorest.info/" target="_blank" title="http://netphorest.info">http://netphorest.info</a></p>
<p>NetPhorest integrates in vitro kinase and phosphopeptide-binding domain specificity assays with publically accessible known in vivo substrate lists in order to generate substrate specificity descriptions for individual proteins as well as protein families.</p>
<p><strong>NetworKIN</strong><br /> Contact: <a href="mailto:rune.linding@gmail.com" title="rune.linding@gmail.com">rune.linding@gmail.com</a><br /> Web Link: <a href="http://networkin.info/" target="_blank" title="http://networkin.info">http://networkin.info</a></p>
<p>NetworKIN predicts the kinase responsible for sites of protein phosphorylation using motif analysis to identify the likely kinase family, and then protein-protein interaction network analysis to identify the most likely individual kinase.</p>
<p><strong>phoMSVal</strong><br /> Contact: <a href="mailto:sampsa.hautaniemi@helsinki.fi" title="sampsa.hautaniemi@helsinki.fi">sampsa.hautaniemi@helsinki.fi</a><br /> Web Link: <a href="http://csbi.ltdk.helsinki.fi/phomsval/" target="_blank" title="http://csbi.ltdk.helsinki.fi/phomsval/">http://csbi.ltdk.helsinki.fi/phomsval/</a></p>
<p>phoMSVal is an open-source platform developed for managing MS/MS data and automatically validating identified phosphopeptides.</p>
<p><strong>PTMScout</strong><br /> Contact: <a href="mailto:ptmscout_admin@mit.edu" title="ptmscout_admin@mit.edu">ptmscout_admin@mit.edu</a><br /> Web Link: <a href="http://ptmscout.mit.edu/" target="_blank" title="http://ptmscout.mit.edu">http://ptmscout.mit.edu</a></p>
<p>PTMScout is a tool for viewing and analyzing high-throughput post-translational modification proteomic data, with a particular eye towards novel hypothesis generation. External data from Gene Ontology, PFAM, and Scansite are integrated with user data automatically, as well as expression data in a large number of tissue types and cell lines. Additionally, all post-translational proteomic mass spectrometry datasets from the White Lab at MIT are banked here, and other users may include their data on request.</p>
<p><strong>Scansite</strong><br /> Contact: Mike Yaffe, Ph.D. (<a href="mailto:myaffe@mit.edu" title="myaffe@mit.edu">myaffe@mit.edu</a>)<br /> Web Link: <a href="http://scansite.mit.edu/" target="_blank" title="http://scansite.mit.edu">http://scansite.mit.edu</a></p>
<p>Scansite contains substrate specificity matrices determined by in vitro peptide library experiments for a number of kinases and phosphopeptide-binding domains. These are used to predict likely interactors with putative substrate sequences of interest.</p>
<p><strong>CellNetOptimizer (CNO)</strong><br /> Contact: <a href="mailto:cno-contact@ebi.ac.uk" title="cno-contact@ebi.ac.uk">cno-contact@ebi.ac.uk</a><br /> Web Link: <a href="http://www.ebi.ac.uk/saezrodriguez/cno/" target="_blank" title="http://www.ebi.ac.uk/saezrodriguez/cno/">http://www.ebi.ac.uk/saezrodriguez/cno/</a></p>
<p>CellNOpt (also known as CNO) is a toolbox for creating logic-based models of signal transduction networks, and training them against high-throughput biochemical data using various logic formalisms. CNO is available both for R and for MATLAB.</p>
<p><strong>Querying Quantitative Logic Models (Q2LM)</strong><br /> Contact: <a href="mailto:morris.melody@gmail.com" title="morris.melody@gmail.com">morris.melody@gmail.com</a>, <a href="mailto:lauffen@mit.edu" title="lauffen@mit.edu">lauffen@mit.edu</a><br /> Web Link: <a href="https://sites.google.com/site/saezrodriguez/software/q2lm" target="_blank" title="https://sites.google.com/site/saezrodriguez/software/q2lm">https://sites.google.com/site/saezrodriguez/software/q2lm</a></p>
<p>Q2LM is a MATLAB library that uses constrained fuzzy logic models based on prior knowledge networks to address two questions about a biological system: 1) What perturbation results in a specified downstream effect? and 2) In what environments do those perturbations produce that effect?</p>
<p><strong>Multiple Clustering Analysis Methodology (MCAM)</strong><br /> Contact: <a href="mailto:knaegle@wustl.edu" title="knaegle@wustl.edu">knaegle@wustl.edu</a><br /> Web Link: <a href="http://naegle.wustl.edu/software.html" target="_blank" title="http://naegle.wustl.edu/software.html">http://naegle.wustl.edu/software.html</a></p>
<p>MCAM is the application of unsupervised learning to high throughput biological datasets of quantitative measurements. Specifically, since the perturbation of data by transformations or changes in the algorithm or distance metric used affects the resulting clustering solution, MCAM seeks to combine large numbers of clustering solutions to better understand the solution space of resulting clusters.</p>
<hr />
<p></p>
<p><strong>Memorial Sloan-Kettering Cancer Center<br />Principal Investigator: Chris Sander, Ph.D.</strong></p>
<p><strong>cBio Cancer Genomics Portal</strong><br /> Contact: <a href="mailto:cbioportal@googlegroups.com" title="cbioportal@googlegroups.com">cbioportal@googlegroups.com</a><br /> Web Link: <a href="http://www.cbioportal.org/" target="_blank" title="http://www.cbioportal.org/">http://www.cbioportal.org/</a></p>
<p>The cBio Cancer Genomics Portal (http://cbioportal.org) is an open-access resource for interactive exploration of multidimensional cancer genomics data sets, currently providing access to data from more than 5,000 tumor samples from 20 cancer studies. The cBio Cancer Genomics Portal significantly lowers the barriers between complex genomic data and cancer researchers who want rapid, intuitive, and high-quality access to molecular profiles and clinical attributes from large-scale cancer genomics projects and empowers researchers to translate these rich data sets into biologic insights and clinical applications.</p>
<p><strong>MEMo</strong><br /> Contact: Giovanni Ciriello (<a href="mailto:ciriello@cbio.mskcc.org" title="ciriello@cbio.mskcc.org">ciriello@cbio.mskcc.org</a>)<br /> Web Link: <a href="http://cbio.mskcc.org/tools/memo/memo.tar.gz" target="_blank" title="http://cbio.mskcc.org/tools/memo/memo.tar.gz">http://cbio.mskcc.org/tools/memo/memo.tar.gz</a></p>
<p>MEMo (Mutually Exclusive Modules) is a method for identifying mutually exclusive driver networks in cancer. The method uses correlation analysis and statistical tests to identify network modules by three criteria: (1) Member genes are recurrently altered across a set of tumor samples; (2) member genes are known to or are likely to participate in the same biological process; and (3) alteration events within the modules are mutually exclusive. The method is freely available as a Java-based command line tool.</p>
<p><strong>Mutation Assessor</strong><br /> Contact: Boris Reva (<a href="mailto:borisr@mskcc.org" title="borisr@mskcc.org">borisr@mskcc.org</a>)<br /> Web Link: <a href="http://mutationassessor.org/" target="_blank" title="http://mutationassessor.org/">http://mutationassessor.org/</a></p>
<p>Mutation Assessor predicts the functional impact of amino-acid substitutions in proteins, such as mutations discovered in cancer or nonsynonymous polymorphisms. The functional impact is assessed based on evolutionary conservation of the affected amino acid in protein homologs. The method has been validated on a large set (51k) of disease associated (OMIM) and polymorphic variants.</p>
<p><strong>NetBox 1.0</strong><br /> Contact: Ethan Cerami (<a href="mailto:cerami@cbio.mskcc.org" title="cerami@cbio.mskcc.org">cerami@cbio.mskcc.org</a>)<br /> Web Link: <a href="http://cbio.mskcc.org/tools/netbox/netbox.tar.gz" target="_blank" title="http://cbio.mskcc.org/tools/netbox/netbox.tar.gz">http://cbio.mskcc.org/tools/netbox/netbox.tar.gz</a></p>
<p>NetBox is a Java-based software tool for performing network analysis on human interaction networks. It is pre-loaded with a Human Interaction Network (HIN) derived from four literature curated data sources, including the Human Protein Reference Database (HPRD), Reactome, NCI-Nature Pathway Interaction (PID) Database, and the MSKCC Cancer Cell Map. It currently supports a command line interface for connecting genes into a network, identifying statistically significant “linker” genes, partitioning the network into modules, and executing two random background models. Results are then made available to the end user as an HTML web page and a series of network and attribute files, which can be loaded into <a href="http://www.cytoscape.org/" target="_blank">Cytoscape</a> for visualization and further analysis.</p>
<p><strong>MicroRNA.org</strong><br /> Contact: <a href="mailto:mirnatargets@cbio.mskcc.org" title="mirnatargets@cbio.mskcc.org">mirnatargets@cbio.mskcc.org</a><br /> Web Link: <a href="http://www.microrna.org/" target="_blank" title="http://www.microrna.org/">http://www.microrna.org/</a></p>
<p>MicroRNA.org is a comprehensive resource of microRNA target predictions and expression profiles. Target predictions are based on a development of the miRanda algorithm, which incorporates current biological knowledge on target rules and on the use of an up-to-date compendium of mammalian microRNAs. MicroRNA target sites are ranked by a down-regulation score using mirSVR, a machine learning method for predicting likelihood of target mRNA down-regulation from sequence and structure features in microRNA/mRNA predicted target sites.</p>
<p><strong>ScatterSlice</strong><br /> Contact: Grégoire Altan-Bonnet (<a href="altanbog%40mskcc.html" title="altanbog@mskcc.org">altanbog@mskcc.org</a>)<br /> Web Link: release in September 2012</p>
<p>Description: ScatterSlice is an R package that applies single-cell heterogeneity analysis (SCHA) on flow cytometry data. This package quantifies automatically the correlation between the phenotypic variability of signaling responses and levels of expression of signaling regulators in isogenic population of cells.</p>
<hr />
<p><strong><br /></strong></p>
<p><strong>Houston Methodist Research Institute<br /> Principal Investigator: Stephen Wong, Ph.D.</strong></p>
<p><strong>Software: NeuriteIQ</strong><br /> Contact email: <a href="mailto:stwong@tmhs.org" title="stwong@tmhs.org">stwong@tmhs.org</a>, <a href="mailto:pshi@tmhs.org" title="pshi@tmhs.org">pshi@tmhs.org</a><br /> Web Link: <a href="http://www.cbi-tmhs.org/NeuriteIQ/index.htm" target="_blank" title="http://www.cbi-tmhs.org/NeuriteIQ/index.htm">http://www.cbi-tmhs.org/NeuriteIQ/index.htm</a></p>
<p>In Alzheimer’s Disease (AD), neurite degeneration is observed in the areas responsible for higher cognitive functions. Loss of neuronal projections in AD can be modeled in vitro in primary mouse cortical neurons treated with the amyloid beta (Aβ) peptide, the primary cause of neurodegeneration in AD. High Content Screening is used to identify compounds specifically suppressing amyloid-induced neurite damage and loss. We developed an integrative image bioinformatics tool, NeuriteIQ to help identify compound hits for treatment of AD and other neurodegenerative diseases. NeuriteIQ can be used for quantitative, reproducible and accurate interpretation of automatic fluorescence microscopy images, in particular, for the labeling and measurement of neurites. It had been evaluated using datasets from Professor Junying Yuan 's Laboratory at Harvard NeuroDiscovery Center, formally known as the HCNR (Harvard Center for Neurodegeneration and Repair), Harvard Medical School; also in collaboration with Professor Alexei Degterev at School of Graduate Biomedical Sciences, Tufts University.</p>
<p><strong>Software: NeuronIQ</strong><br /> Contact email: <a href="mailto:stwong@tmhs.org" title="stwong@tmhs.org">stwong@tmhs.org</a>, <a href="mailto:jcheng@tmhs.org" title="jcheng@tmhs.org">jcheng@tmhs.org</a><br /> Web Link: <a href="http://www.cbi-tmhs.org/Neuroniq/index.htm" target="_blank" title="http://www.cbi-tmhs.org/Neuroniq/index.htm">http://www.cbi-tmhs.org/Neuroniq/index.htm</a></p>
<p>Confocal and two-photon microscopy can image a neuronal specimen in 3D and produce rich information about the fine structures, such as neuronal dendrites and spines, in high resolution. Manual analysis is slow, and many time-lapse experiments are limited to small-scale manual analysis. The bottleneck to efficiently and effectively exploit the data is the lack of a computer-based system to track, analyze, quantitate, and store neuron images. We proposed to develop a computational neurobiology system, t-NeuronlQ (time-lapse Neuron image Quantitator) to track, analyze, and store neuronal dendrite and spine changes in time-lapse 3D optical microscopy images. The system will automatically and semi-automatically track and analyze dendrites and spines in a time-lapse manner. We developed software packages, called NeuronIQ for segmenting and tracking spine and dendrites in 2D maximum projection neuron images, volumetric 3D images, and time-lapse 3D neuron images. The 2D NeuronIQ had been evaluated using datasets from Dr. Yong Kim in Rockefeller University. We are currently conducting an extensive evaluation of the 3D NeuronIQ and t-NeuronIQ by collaborating with Dr. Kim and Dr. Thomas A. Blanpied, Department of Physiology, University of Maryland School of Medicine.</p>
<p><strong>Dynamic Cell Image Quantitator (DCellIQ)</strong><br /> Contact: Stephen Wong, Ph.D. (<a href="mailto:stwong@tmhs.org" title="stwong@tmhs.org">stwong@tmhs.org</a>)<br /> Web Link: <a href="http://www.cbi-tmhs.org/Dcelliq/index.html" target="_blank" title="http://www.cbi-tmhs.org/Dcelliq/index.html">http://www.cbi-tmhs.org/Dcelliq/index.html</a></p>
<p>The software aims to provide an automated pipeline for quantitative, reproducible and accurate interpretation of cell dynamic behaviors using time-lapse cellular images. DCellIQ is an automated tool with several parameters being set before the processing. Thus, it is well suitable for batch processing large image dataset with little human interference. This work was done in large part while at the Harvard Center for Neurodegeneration and Repair.</p>
<p><strong>Genomic Cell Image Quantitator (GCellIQ)</strong><br /> Contact: Stephen Wong, Ph.D. (<a href="mailto:stwong@tmhs.org" title="stwong@tmhs.org">stwong@tmhs.org</a>)<br /> Web Link: <a href="http://www.cbi-tmhs.org/GCellIQ/" target="_blank" title="http://www.cbi-tmhs.org/GCellIQ/">http://www.cbi-tmhs.org/GCellIQ/</a></p>
<p>The software aims to provide an automated pipeline for processing large volumes of digital images generated from genome-wide, high-content screening (HCS), including those by RNA interference (RNAi) or other small molecular screenings. GCellIQ is an automated tool with several parameters being set before the processing. Thus, it is well suitable for batch processing large image dataset with little human interference. The bulk of the development effort was done in close collaboration with Perrimon's Laboratory at Howard Hughes Medical Institute, Harvard Medical School.</p>
<p><strong>Zebrafish Image Quantitator (ZFIQ)</strong><br /> Contact: Stephen Wong, Ph.D. (<a href="mailto:stwong@tmhs.org" title="stwong@tmhs.org">stwong@tmhs.org</a>)<br /> Web Link: <a href="http://www.cbi-tmhs.org/ZFIQ/index.htm" target="_blank" title="http://www.cbi-tmhs.org/ZFIQ/index.htm">http://www.cbi-tmhs.org/ZFIQ/index.htm</a></p>
<p>The <a href="http://www.cbi-tmhs.org/ZFIQ/download.htm" target="_blank" title="ZFIQ">ZFIQ</a> (Zebrafish Image Quantitator) software toolkit is developed by the Medical Image Computing Group of the Center for Bioengineering and Informatics (CBI, Director: Dr. Stephen Wong), Houston Methodist Research Institute with a large part of the development effort was done while Stephen’s group was part of the <a href="http://www.neurodiscovery.harvard.edu/" target="_blank" title="Harvard NeuroDiscovery Center">Harvard NeuroDiscovery Center</a>, formally known as the HCNR (Harvard Center for Neurodegeneration and Repair), Harvard Medical School. It is a collaborative effort between Weiming Xia, Department of Neurology, Brigham and Women's Hospital, Harvard Medical School; Jarema Malicki, Department of Ophthalmology, Massachusetts Eye and Ear Infirmary, Harvard Medical School; and Scott Holley, Department of Molecular, Cellular and Developmental Biology, Yale University. The software aims to provide a set of practical and effective image analysis tools for quantitative, reproducible and accurate interpretation of zebrafish imaging data, in particular, for streamlined processing and analysis of bioimaging data. The design of ZFIQ is modular such that new applications and modules can be added into the toolkit.</p>
<p><strong>Axon Tracker</strong><br /> Contact: Stephen Wong, Ph.D. (<a href="mailto:stwong@tmhs.org" title="stwong@tmhs.org">stwong@tmhs.org</a>)<br /> Web Link: <a href="http://www.cbi-tmhs.org/AxonTracker/" target="_blank" title="http://www.cbi-tmhs.org/AxonTracker/">http://www.cbi-tmhs.org/AxonTracker/</a></p>
<p>AxonTracker 3D software tool was developed by the Center for Bioengineering and Informatics (CBI) at the Houston Methodist Research Institute in collaboration with Lichtman Group at Department of Molecular and Cellular Biology, Harvard University. The software is useful to semi-automatically track and label axon structures from the motor axon connectome datasets. It is a parameter-free software, and hence can be used over a broad range of datasets. The software is designed with ease of use in mind. The interface is simple and includes a 3D visualization of the dataset along with a real-time overlay of the tracking results. A 2D view of the dataset is also provided in order to interact with the software. Three orthogonal views are provided with a provision for back-tracking the results. Points can be added/deleted from the tracking results to provide guidance in case of an error.</p>
<p><strong>Assisted Follow-up in NeuroImaging of Therapeutic Intervention (AFINITI)</strong><br /> Contact: Stephen Wong, Ph.D. (<a href="mailto:stwong@tmhs.org" title="stwong@tmhs.org">stwong@tmhs.org</a>)<br /> Web Link: <a href="http://www.cbi-tmhs.org/AFINITI/" target="_blank" title="http://www.cbi-tmhs.org/AFINITI/">http://www.cbi-tmhs.org/AFINITI/</a></p>
<p>AFINITI (Assisted Follow-up in NeuroImaging of Therapeutic Intervention) is translational image informatics system designed to enhance efficacy and reduce error in the interpretation of neuroimaging follow-up studies for several common indications, by providing the interpreting and referring physicians with automated tools and augmented information needed to allow sensitive, reproducible, quantitative assessment of longitudinal changes in brains in response to therapeutic intervention. The primary goal is to introduce the tools and techniques resulting from biomedical informatics research into real-life settings in clinical care, education and basic biomedical research of neurological diseases. The secondary goal is to design and offer a new information service to support neuroimaging follow-up studies. The software package integrates the medical image processing pipelines for semi-automatic and sensitive follow up of longitudinal changes in human brain. It is developed on top of ITK-SNAP version 1.5.4, and most ITK-SNAP features have been kept. It runs in Windows environment, and does not require an installation step. To run it, just extract the zipped program file to a folder on your workstation, and double click the executable file in the top folder.</p>
<hr />
<p><strong><br /></strong></p>
<p><strong>Oregon Health &amp; Science University<br /> Principal Investigator: Joe Gray, Ph.D.</strong></p>
<p>None</p>
<hr />
<p><strong><br /></strong></p>
<p><strong>Sage Bionetworks<br /> Principal Investigator: Stephen Friend, M.D., Ph.D.</strong></p>
<p><strong>SYNAPSE</strong><br /> Conatct: <a href="mailto:synapseInfo@sagebase.org" title="synapseInfo@sagebase.org">synapseInfo@sagebase.org</a><br /> Web Link: <a href="https://synapse.sagebase.org/ - Home:null" target="_blank" title="https://synapse.sagebase.org/ - Home:null">https://synapse.sagebase.org/ - Home:null</a></p>
<p>Synapse is an innovation compute space that brings together scientific data, tools, and disease models into a Commons that enables true collaborative research for those who want to share and analyze data together. The platform consists of a web portal, web services, and integrations with data analysis tool and is organized around novel “Analysis Communities” that any scientist can create or join to work on complex medical problems. In Synapse you can:</p>
<ul>
<li>Search across public data including curated GEO, TCGA, and Array Express</li>
<li>Find raw data or follow an analysis path to processed results</li>
<li>Load data directly into analytical tools like R, and record your analysis</li>
<li>Create Public or Private projects to share your work</li>
</ul>
<p>Synapse is designed to enable many layers of collaboration: within a single team, between research teams, between research teams and industry partners, and within pharmaceutical companies. An example of activities in Synapse is the Sage / DREAM Breast Cancer Prognosis Challenge. More similar challenges can be executed in the Synapse environment.</p>
<p><strong>Key Driver Analysis (KDA)</strong><br /> Conatct: <a href="mailto:tools@sagebase.org" title="tools@sagebase.org">tools@sagebase.org</a>.<br /> Web Link: <a href="https://sourceforge.net/projects/sagebionetworks/files" target="_blank" title="https://sourceforge.net/projects/sagebionetworks/files">https://sourceforge.net/projects/sagebionetworks/files</a> to download KDA Java Plugins and R Package</p>
<p>Key Driver Analysis is an analysis tool, as both an R package and Cytoscape plugin, for identifying key regulators of a gene regulatory network. It takes as input a gene network N (directed or undirected) and a gene set (module) G. The gene set is any subset of genes from the network N (e.g. pathway, module, ontology), permitting focus on a particular biological context.</p>
<hr />
<p><strong><br /></strong></p>
<p><strong>St. Elizabeth’s Medical Center/Tufts University Principal Investigator: Lynn Hlatky, Ph.D.</strong></p>
<p><strong>SUMO</strong><br /> Contact: Heiko Enderling, Ph.D. (<a href="mailto:heiko.enderling@tufts.edu" title="heiko.enderling@tufts.edu">heiko.enderling@tufts.edu</a>)<br /> Web Link: <a href="http://angiogenesis.dkfz.de/oncoexpress/software/index.htm" target="_blank">http://angiogenesis.dkfz.de/oncoexpress/software/index.htm</a></p>
<p>A gene expression and data analysis tool.</p>
<p><strong>Cell-Type Tumor Contribution Model?</strong><br /> Contact: Heiko Enderling, Ph.D. (<a href="mailto:heiko.enderling@tufts.edu" title="heiko.enderling@tufts.edu">heiko.enderling@tufts.edu</a>)<br /> Web Link: <a href="http://www.cancer-systems-biology.org/index.html?/ccsb/memberpages/heiko/abm.html" target="_blank" title="http://www.cancer-systems-biology.org/index.html?/ccsb/memberpages/heiko/abm.html">http://www.cancer-systems-biology.org/index.html?/ccsb/memberpages/heiko/abm.html</a></p>
<p>An agent-based model of early tumor growth dynamics.</p>
<p><strong>CompuCell3D cell tracking module</strong><br /> Contact: Heiko Enderling, Ph.D. (<a href="mailto:heiko.enderling@tufts.edu" title="heiko.enderling@tufts.edu">heiko.enderling@tufts.edu</a>)</p>
<p>A module for CompuCell3D to track migration of in silico cells.</p>
<hr />
<p><strong><br /></strong></p>
<p><strong>University of Texas Health Science Center, San Antonio Principal Investigator: Tim H-M Huang, Ph.D.</strong></p>
<p><strong>Pathway distiller</strong><br /> Contact: Mark Doderer, Ph.D. (<a href="mailto:Doderer@uthscsa.edu" title="Doderer@uthscsa.edu">Doderer@uthscsa.edu</a>)<br /> Web Link: <a href="http://cbbiweb.uthscsa.edu/PathwayDistiller" target="_blank" title="http://cbbiweb.uthscsa.edu/PathwayDistiller">http://cbbiweb.uthscsa.edu/PathwayDistiller</a></p>
<p>Pathway distiller combines several pathway systems and implements different, but complementary pathway consolidation methods. It provides a user-friendly web-accessible tool that enables users the ability to extract functional explanations of their genome wide experiments.</p>
<p><strong>CommFinder</strong><br /> Contact: Jianhua Ruan, Ph.D. (<a href="mailto:jruan@cs.utsa.edu" title="jruan@cs.utsa.edu">jruan@cs.utsa.edu</a>) <br /> Web Link: <a href="http://cs.utsa.edu/~jruan/software/commFinder/CommFinder.htm" target="_blank" title="http://cs.utsa.edu/~jruan/software/commFinder/CommFinder.htm">http://cs.utsa.edu/~jruan/software/commFinder/CommFinder.htm</a></p>
<p>CommFinder is a Cytoscape plugin for finding and visualizing community structures in biological networks. It provides interfaces to several community finding algorithms including Qcut, HQcut, MCL and MCODE.</p>
<p><strong>Qcut/HQcut</strong><br /> Contact: Jianhua Ruan, Ph.D. (<a href="mailto:jruan@cs.utsa.edu" title="jruan@cs.utsa.edu">jruan@cs.utsa.edu</a>)<br /> Web Link: <a href="http://cs.utsa.edu/~jruan/Software.html" target="_blank" title="http://cs.utsa.edu/~jruan/Software.html">http://cs.utsa.edu/~jruan/Software.html</a></p>
<p>This includes a set of MATLAB scripts for finding community structures in biological networks. Qcut is an algorithm for finding community structures based on modularity optimization. HQcut is an improvement of Qcut and solves the well-known resolution limit problem of the modularity function. The package also includes some scripts for visualizing network communities, and for constructing and clustering gene co-expression networks.</p>
<p><strong>BDTree</strong><br /> Contact: Jianhua Ruan, Ph.D. (<a href="mailto:jruan@cs.utsa.edu" title="jruan@cs.utsa.edu">jruan@cs.utsa.edu</a>)<br /> Web Link: <a href="http://cs.utsa.edu/~jruan/Software.html" target="_blank" title="http://cs.utsa.edu/~jruan/Software.html">http://cs.utsa.edu/~jruan/Software.html</a></p>
<p>A bi-dimensional regression tree approach for modeling transcriptional regulation. The model associates transcription factor binding sites and expression levels of transcription factors with gene expression levels of target genes.</p>
<p><strong>ILM</strong><br /> Contact: Jianhua Ruan, Ph.D. (<a href="mailto:jruan@cs.utsa.edu" title="jruan@cs.utsa.edu">jruan@cs.utsa.edu</a>)<br /> Web Link: <a href="http://cs.utsa.edu/~jruan/Software.html" target="_blank" title="http://cs.utsa.edu/~jruan/Software.html">http://cs.utsa.edu/~jruan/Software.html</a></p>
<p>This software implements the Iterated Loop Matching algorithm for predicting pseudoknotted RNA secondary structures for a single RNA sequence or multiple aligned RNA sequences.</p>
<p><strong>BCmicrO</strong><br /> Contact: Yufei Huang, Ph.D. (<a href="mailto:yhuang@utsa.edu" title="yhuang@utsa.edu">yhuang@utsa.edu</a>) <br /> Web Link: <a href="http://compgenomics.utsa.edu/gene/gene_1.php" target="_blank" title="http://compgenomics.utsa.edu/gene/gene_1.php">http://compgenomics.utsa.edu/gene/gene_1.php</a></p>
<p>BCmicrO is an algorithm that combines the microRNA predictions of 6 existing algorithms including TargetScan, miRanda, PicTar, mirTarget, PITA, DianamicroT using a Bayesian Network.</p>
<p><strong>SVMicrO</strong><br /> Contact: Yufei Huang, Ph.D. (<a href="mailto:yhuang@utsa.edu" title="yhuang@utsa.edu">yhuang@utsa.edu</a>) <br /> Web Link: <a href="http://compgenomics.utsa.edu/svmicro.html" target="_blank" title="http://compgenomics.utsa.edu/svmicro.html">http://compgenomics.utsa.edu/svmicro.html</a></p>
<p>SVMmicrO is a microRNA target prediction algorithm that utilizes a comprehensive microRNA binding features.</p>
<p><strong>QUEST</strong><br /> Contact: Kun Huang, Ph.D. (<a href="mailto:Kun.Huang@osumc.edu" title="Kun.Huang@osumc.edu">Kun.Huang@osumc.edu</a>)<br /> Web Link: <a href="https://bisr.osumc.edu/QUEST/Public/Login.aspx?ReturnUrl=%2fQUEST%2fStartPage.aspx" target="_blank" title="https://bisr.osumc.edu/QUEST/Public/Login.aspx?ReturnUrl=%2fQUEST%2fStartPage.aspx">https://bisr.osumc.edu/QUEST/Public/Login.aspx?ReturnUrl=%2fQUEST%2fStartPage.aspx</a></p>
<p>This is a next-generation sequencing data management and query tool.</p>
<p><strong>W-ChIPMotifs</strong><br /> Contact: Victor Jin, Ph.D. (<a href="mailto:Victor.Jin@osumc.edu" title="Victor.Jin@osumc.edu">Victor.Jin@osumc.edu</a>)<br /> Web Link: <a href="http://motif.bmi.ohio-state.edu/ChIPMotifs/" target="_blank" title="http://motif.bmi.ohio-state.edu/ChIPMotifs/">http://motif.bmi.ohio-state.edu/ChIPMotifs/</a></p>
<p>A Web application tool for de novo motif discovery from ChIP-based high throughput data.</p>
<p><strong>MyTrack</strong><br /> Contact: Yunlong Liu, Ph.D. (<a href="mailto:yunliu@iupui.edu" title="yunliu@iupui.edu">yunliu@iupui.edu</a>)<br /> Web Link: <a href="http://watson.compbio.iupui.edu/mytrack" target="_blank" title="http://watson.compbio.iupui.edu/mytrack">http://watson.compbio.iupui.edu/mytrack</a></p>
<p>Key Driver Analysis (KDA) is an analysis tool, as both an R package and Cytoscape plugin, for identifying key regulators of a gene regulatory network. It takes as input a gene network N (directed or undirected) and a gene set (module) G. The gene set is any subset of genes from the network N (e.g. pathway, module, ontology), permitting focus on a particular biological context.</p>
<p><strong>DIME</strong><br /> Contact: Shili Lin, Ph.D. (<a href="mailto:shili@stat.osu.edu" title="shili@stat.osu.edu">shili@stat.osu.edu</a>)<br /> Web Link: <a href="http://www.stat.osu.edu/~statgen/SOFTWARE/DIME/" target="_blank" title="http://www.stat.osu.edu/~statgen/SOFTWARE/DIME/">http://www.stat.osu.edu/~statgen/SOFTWARE/DIME/</a></p>
<p>An ensemble of methods for differential analysis. Specifically, it considers an ensemble of finite mixture models combined with a local false discovery rate for analyzing ChIP-seq data comparing two samples. This package can also be used to identify differential in other high throughput data such as microarray and DNA methylation.</p>
<hr />
<p><strong><br /></strong></p>
<p><strong>Vanderbilt University Medical Center <br /> Principal Investigator: Vito Quaranta, M.D.</strong></p>
<p><strong>Cell Animation</strong><br /> Web Link: <a href="http://ccsb.vanderbilt.edu/ccsb/CellAnimation" target="_blank" title="http://ccsb.vanderbilt.edu/ccsb/CellAnimation">http://ccsb.vanderbilt.edu/ccsb/CellAnimation</a></p>
<p><strong>(under construction)</strong></p>
<p>Cell Animation is an open source Matlab framework for analyzing image stacks. It features several GUI interfaces to aid in the construction of an image processing assy. It comes with several assays that perform functions such as cell tracking, ancestry tree construction, cell spreading, focal adhesion turn over, cytoplasm segmentation, and colony counting. Assay parameters are stored separately from algorithm, so that a given assay can be used with multiple configurations.</p>
<p><strong>R Package for Fractional Proliferation</strong><br /> Web Link: <a href="http://cran.r-project.org/web/packages/fracprolif/index.html" target="_blank" title="http://cran.r-project.org/web/packages/fracprolif/index.html">http://cran.r-project.org/web/packages/fracprolif/index.html</a></p>
<p>‘fracprolif' A statistical package to estimate and predict growth based on division, death and quiescence is provided through CRAN as part of the R environment.</p>
<p><strong>Mathematical Models</strong><br /> Contact: Alexander "Sandy" Anderson, Ph.D. (<a href="mailto:alexander.anderson@moffitt.org" title="alexander.anderson@moffitt.org">alexander.anderson@moffitt.org</a>)<br /> Web Link: <a href="http://ccsb.vanderbilt.edu/ccsb/math_models" target="_blank" title="http://ccsb.vanderbilt.edu/ccsb/math_models">http://ccsb.vanderbilt.edu/ccsb/math_models</a></p>
<p>Various mathematical models developed by the Center for Cancer Systems Biology at Vanderbilt University.</p>
            
        </div>
    
    

                                 </div>
    
                                 <div id="viewlet-below-content-body">



    <div class="visualClear"><!-- --></div>

    <div class="documentActions">
        

        

    </div>
</div>
    
                                
                            </div>
                        </td>
                        <td>
                            
                                
                            
                        </td>
                    </tr>
                    </table>
                    


                    
                </div>
            

            <div id="viewlet-below-content">





</div>
        </div>


    </div>


    <div class="row">
        <div class="subpage-footer">
           <div class="row">
<div class="cell width-full position-0">
        <div id="portal-footer"><div class="left"></div>
		<div class="footerLinks"><a href="../contact_us.html" title="Contact Us">Contact Us</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www.nih.gov/about/privacy.htm" title="Privacy Notice">Privacy Notice</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www.nih.gov/about/disclaim.htm" title="Disclaimer">Disclaimer</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://www3.cancer.gov/accessibility/nci508.htm" title="Accessibility">Accessibility</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://cancer.gov/global/viewing-files" target="_blank" title="Viewing Files">Viewing Files</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://ncicb.nci.nih.gov/support/" target="_blank" title="Application Support">Application Support</a></div>
       <div id="org"> 

			<div align="center">
<a href="http://www.dhhs.gov/"><img title="Department of Health and Human Services" src="../footer_hhs.gif" width="45" height="43" alt="Department of Health and Human Services" border="0"></a>
<a href="http://www.nih.gov/"><img title="National Institutes of Health" src="../footer_nih.gif" width="45" height="43" alt="National Institutes of Health" border="0"></a>
<a href="http://www.cancer.gov/"><img title="National Cancer Institute" src="../footer_nci.gif" width="57" height="43" alt="National Cancer Institute" border="0"></a>
<a href="http://www.usa.gov/"><img title="USA.gov" src="../footer_usagov.gif" width="93" height="43" alt="USA.gov" border="0"></a>
			</div>
<br /><br /><br />NIH...Turning Discovery Into Health

       </div>
        </div>



    </div>
</div>

        </div>
        <div id="kss-spinner">
            <img alt="" src="../spinner.gif" />
        </div>
    </div>

</div>
</body>

<!-- Mirrored from icbp.nci.nih.gov/resources/software by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 29 Sep 2016 19:22:26 GMT -->
</html>



